---
title: Data
layout: data
redirect_to:
  - http://tissue-atlas.org/data-overview
---
<h2 class="h2">
  Visualizing and sharing multiplexed CyCIF data
</h2>
<p class="">
  For a ~10 x 11mm tissue FFPE specimen under 20X objective, each cycle of t-CyCIF generates around 160 individual
  image tiles.
  The assembly process involves stitching sequential image tiles from a single t-CyCIF cycle into one large image
  panel,
  flat-fielding to correct for uneven illumination and registration of images from successive t-CyCIF cycles to each
  other;
  these procedures were performed using
  <a href="https://fiji.sc/" target="_blank">ImageJ</a>,
  <a href="https://github.com/sorgerlab/ashlar" target="_blank">ASHLAR</a>, and
  <a href="https://www.nature.com/articles/ncomms14836" target="_blank">BaSiC software</a>.
</p>
<p class="mb-5">
    These sharable, guided analyses are created by users of all levels of coding experience using the <a href="{{ site.baseurl }}/software/minerva/">Minerva suite</a>. This includes Minerva Author, a tool that enables the user to easily create a guided analysis and Minerva Story, a tool that enables users to turn their story into a narrative, web-based image for viewing and sharing. 
</p>
<div class="mb-5">
    <select class="select2-search" name="figures">
    </select>
</div>
<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/cycifviewer/pca-2019.jpg" alt="Generic placeholder image">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/data/pca-2019/">
      <h5 class="mt-0">Melanoma Pre-Cancer Atlas (HTAN)</h5>
    </a>
    <p>
      Unpublished t-CyCIF images of melanoma and precursor fields from two patients. The specimen from patient 1 illustrates different regions of melanoma progression from relatively normal melanocytes to precursor melanocytic dysplasia to invasive melanoma. In addition, the specimen shows different regions representing immune responses to early melanoma (inflammatory regression) as well as to invasive melanoma in the form of a brisk immune infiltrative immune response (tumor infiltrating lymphocytes “TILs”). The excision specimen from patient 2 illustrates the histologic evolution of melanoma from a precursor field to melanoma in situ, and ultimately to polypoidal invasive melanoma.
    </p>
  </div>
</div>

<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/cycifviewer/01_melanoma_story.jpg" alt="Generic placeholder image">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/osd-exhibit" class="">
      <h5 class="mt-0">Tour of the melanoma dataset</h5>
    </a>
    <p>
      The unpublished 13-plex t-CyCIF images of tissue specimens from four patients with BRAF-mutant metastatic
      melanoma resected
      before (left) and after (right) treatment with BRAF and MEK inhibitors (dabrafenib/trametinib). Each image is
      composed
      of 150-200 image tiles at a nominal resolution of ~0.9 μm.
    </p>
  </div>
</div>

<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/cycifviewer/02_square_elife_paper.jpg" alt="Generic placeholder image">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/data/lin-elife-2018/">
      <h5 class="mt-0">Figures in the 2018 eLife Publication</h5>
    </a>
    <p>
      Tissue-based cyclic immunofluorescent microscopy (t-CyCIF) is a simple method for generating highly multiplexed
      optical images
      from formalin-fixed paraffin-embedded (FFPE) tissue samples routinely used for histopathological diagnosis of
      human
      disease. The method is based on previously described single-cell imaging approaches and readily implemented on
      existing
      instruments (Gerdes et al. 2013, Lin et al. 2015, 2016).
    </p>
  </div>
</div>

<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/cycifviewer/03_gaglia.jpg" alt="HFS1">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/featured-paper/gaglia-2018/figures/osd-COLON_1">
      <h5 class="mt-0">Gaglia et al., Nature Cell Biology, 2020</h5>
    </a>
    <p>
			<a href="https://www.synapse.org/#!Synapse:syn20505972/wiki/594014">
			Access Raw Data
			</a>
    </p>
  </div>
</div>
<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/du-lin-rashid-nat-protoc-2019/TONSIL_1.jpg" alt="nat_prot">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/data/du-lin-rashid-nat-protoc-2019/">
      <h5 class="mt-0">Du, Lin, and Rashid et al., Nature Protocols, 2019</h5>
    </a>
    <p>

    </p>
  </div>
</div>
<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/liu-lin-2019/cover.jpg" alt="nat_medicine">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/data/liu-lin-2019/">
      <h5 class="mt-0">Liu and Lin et al., Nature Medicine (accepted)</h5>
    </a>
    <p>
      Despite initial responses, most melanoma patients develop resistance to immune checkpoint blockade (ICB) via poorly understood mechanisms. To understand the evolution of resistance, we studied 37 tumor samples collected over 9 years from a patient with metastatic melanoma and initial response to ICB followed by delayed recurrence and death. Phylogenetic analysis revealed co-evolution of 7 lineages with multiple convergent, but independent resistance-associated alterations (RAAs). All progression tumors emerged from a lineage characterized by loss of chromosome 15q including β2 microglobulin (B2M), with post-treatment resistant clones continuing to acquire additional genomic driver events including genome doubling, CDKN2A biallelic loss, and chromosome 11 loss. Deconvolution of bulk RNAseq and highly-multiplexed immunofluorescence (t-CyCIF) of histological specimens revealed differences in immune composition amongst the different lineages, despite an overall low immune composition. Imaging also revealed an NGFR-High tumor population enriched for PD-L1 expression in close proximity to immune cells with a vasculogenic mimicry phenotype. Rapid autopsy samples demonstrated 2 distinct NGFR spatial patterns with increased polarity and proximity to immune cells in subcutaneous tumors versus a diffuse spatial pattern in lung tumors, suggesting differences in the role and etiology of this neural crest-like program in different tumor microenvironments.
    </p>
  </div>
</div>
<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/mehta-2020/cover.jpg" alt="nat_medicine">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/data/mehta-2020/">
      <h5 class="mt-0">Mehta et al., Nature Cancer, 2020</h5>
    </a>
    <p>
      Despite objective responses to PARP inhibition and improvements in progression-free survival compared to standard chemotherapy in patients with BRCA-associated triple-negative breast cancer (TNBC), benefits are transitory. Using high dimensional single-cell profiling of human TNBC, here we demonstrate that macrophages are the predominant infiltrating immune cell type in BRCA-associated TNBC. Through multi-omics profiling we show that PARP inhibitors enhance both anti- and pro-tumor features of macrophages through glucose and lipid metabolic reprogramming driven by the sterol regulatory element-binding protein 1 (SREBP-1) pathway. Combined PARP inhibitor therapy with CSF-1R blocking antibodies significantly enhanced innate and adaptive anti-tumor immunity and extends survival in BRCA-deficient tumors in vivo and is mediated by CD8+ T-cells. Collectively, our results uncover macrophage-mediated immune suppression as a liability of PARP inhibitor treatment and demonstrate combined PARP inhibition and macrophage targeting therapy induces a durable reprogramming of the tumor microenvironment, thus constituting a promising therapeutic strategy for TNBC.
    </p>
  </div>
</div>

<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/keenan-2020/cover.jpg" alt="Thumbnail for Keenan-2020">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/data/keenan-2020/">
      <h5 class="mt-0">Keenan et al., 2020 (in preparation)</h5>
    </a>
    <p>
      Selective inhibition of the negative cell cycle regulator WEE1 may enhance the efficacy of DNA-damaging agents by reducing DNA damage repair. These are multiplexed cyclic immunofluorescence on paired pre- and post-WEE1 inhibitor tumor biopsies, from the first phase II study assessing the efficacy of the WEE1 inhibitor adavosertib with cisplatin in metastatic triple-negative breast cancer (mTNBC). Among patients with mTNBC treated with 0-1 prior lines, adavosertib combined with cisplatin narrowly missed the prespecified ORR cutoff of > 30%. The finding of immune infiltrated tumors in patients with clinical benefit to therapy requires validation in future studies.
    </p>
  </div>
</div>

<div class="row mb-4">
  <div class="col-md-2">
    <img class="img-fluid mb-3 w-100" src="{{ site.baseurl }}/assets/img/gaglia-mit-ludwig-2020/gemm-lung.jpg" alt="Thumbnail for gaglia-mit-ludwig-2020">
  </div>
  <div class="col">
    <a href="{{ site.baseurl }}/data/gaglia-mit-ludwig-2020/">
      <h5 class="mt-0">Spatio-temporal effects of genetic and therapeutic perturbation on the immune micro-environment in murine models of lung and pancreas adenocarcinomas</h5>
    </a>
    <p>
      In a collaboration with Dr. Tyler Jacks at Ludwig MIT, we used t-CyCIF to characterize tumor intrinsic and extrinsic states in genetically engineered cancer models of lung and pancreas samples treated with clinical trial agents such as HSP90 inhibitors, classic cytotoxic agents, and immune checkpoint blockade modifiers. We are interested in understanding the effects of genetic perturbations, for example targeted deletion or over-expression of individual or panels of cancer, and immune-related genes in these samples.
    </p>
  </div>
</div>
