<!DOCTYPE html>
<html lang="en-US" class="h-100">

<head>
  <meta charset='utf-8'>
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="/assets/css/style.css?v="> <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>CRC01 - Intro - Lin, Wang, Coy et al, 2021 | CyCIF</title>
<meta name="generator" content="Jekyll v3.9.3" />
<meta property="og:title" content="CRC01 - Intro - Lin, Wang, Coy et al, 2021" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="A Simple Approach to Highly Multiplexed Cell and Tissue Imaging" />
<meta property="og:description" content="A Simple Approach to Highly Multiplexed Cell and Tissue Imaging" />
<meta property="og:site_name" content="CyCIF" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="CRC01 - Intro - Lin, Wang, Coy et al, 2021" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","description":"A Simple Approach to Highly Multiplexed Cell and Tissue Imaging","headline":"CRC01 - Intro - Lin, Wang, Coy et al, 2021","url":"/data/lin-wang-coy-2021/overview.html"}</script>
<!-- End Jekyll SEO tag -->


</head>

<body class="d-flex flex-column h-100">
    <!-- style="position:relative" -->

  
  

  
  

  
  
  

  
  

    <header class="m-0">
        <nav class="main-nav navbar navbar-expand-lg navbar-dark fixed-top bg-dark mb-md-4">
  <div class="container">
    <a class="navbar-brand" href="/">
      <strong>CyCIF</strong>
    </a>
    <button class="navbar-toggler p-2" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault"
      aria-expanded="false" aria-label="Toggle navigation">
      <span class="text-white">MENU</span>
    </button>
    <div class="collapse navbar-collapse" id="navbarsExampleDefault">
      <ul class="navbar-nav mr-auto">
        
        <li class="nav-item">
          <a href="/data" class="nav-link  active " style="font-size: 1rem">
            Data
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/methods" class="nav-link " style="font-size: 1rem">
            Methods
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/antibodies" class="nav-link " style="font-size: 1rem">
            Antibodies
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/software" class="nav-link " style="font-size: 1rem">
            Software
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/publications.html" class="nav-link " style="font-size: 1rem">
            Publications
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/people.html" class="nav-link " style="font-size: 1rem">
            People
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/funding" class="nav-link " style="font-size: 1rem">
            Funding
          </a>
        </li>
        
      </ul>
          <div class="flex-row ml-md-auto">
  
            <a href="/featured-paper/du-lin-rashid-2019/figures/osd-LUNG_3" class="btn btn-outline-light font-weight-bold" style="font-size: 1rem; background: url(/assets/img/explore_button.jpg) center; background-size: cover;">
              <span class="text-white">
                  Explore CyCIF
              </span> 
            </a>
          </div> 
    </div>
  </div>
</nav>
<nav class="navbar navbar-expand-md navbar-dark bg-dark">
  <div class="container">
    <a class="navbar-brand" href="#">
      <strong>CyCIF</strong>
    </a>
  </div>
</nav>

    </header>

    <div style="height:100%" id="minerva-browser"></div>

    <script defer src="https://use.fontawesome.com/releases/v5.2.0/js/all.js" integrity="sha384-4oV5EgaV02iISL2ban6c/RmotsABqE4yZxZLcYMAdG7FAPsyHYAPpywE9PJo+Khy" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script> 
    <script src="https://api.html5media.info/1.2.2/html5media.min.js"></script>

    <script src="https://cdn.jsdelivr.net/npm/minerva-browser@3.2.0/build/bundle.js"></script>
    <script>
        window.viewer = MinervaStory.default.build_page({
          embedded: true,
          id: "minerva-browser",
          exhibit: {"Name":"Multiplexed 3D atlas of state transitions and immune interactions in colorectal cancer (Introduction)\n","Groups":[{"Channels":["Hematoxylin","Eosin"],"Colors":["8233CC","FF8080"],"Name":"Histology","Path":"Histology_40__HE-r--41__HE-g--42__HE-b"},{"Channels":["DNA1","PanCK","ASMA","CD31","CD45"],"Colors":["0000ff","ffffff","ff0000","00ff00","ffff00"],"Name":"Tissue Structure","Path":"Tissue-Structure_0__DNA1--14__PanCK--15__ASMA--35__CD31--18__CD45"},{"Channels":["DNA1","CD45","CD8a","CD4","CD20","CD68","CD163"],"Colors":["0000ff","ffffff","ff0000","ff00ff","ffff00","00ff00","00ffff"],"Name":"Immune Populations","Path":"Immune-Populations_0__DNA1--18__CD45--23__CD8a--17__CD4--21__CD20--22__CD68--25__CD163"},{"Channels":["DNA1","CD8a","CD4","CD20","FOXP3"],"Colors":["0000ff","00ff00","ff0000","ffff00","ffffff"],"Name":"Lymphocytes","Path":"Lymphocytes_0__DNA1--23__CD8a--17__CD4--21__CD20--26__FOXP3"},{"Channels":["DNA1","CD68","CD163"],"Colors":["0000ff","00ff00","ff0000"],"Name":"Macrophages","Path":"Macrophages_0__DNA1--22__CD68--25__CD163"},{"Channels":["DNA1","PCNA","PanCK","CD45","Ki67"],"Colors":["0000ff","ff0000","ffffff","00ffff","00ff00"],"Name":"Proliferation","Path":"Proliferation_0__DNA1--37__PCNA--14__PanCK--18__CD45--13__Ki67"},{"Channels":["DNA1","PanCK","PDL1","PD1"],"Colors":["0000ff","ffffff","ff0000","00ff00"],"Name":"PD1 Immune Checkpoint","Path":"PD1-Immune-Checkpoint_0__DNA1--14__PanCK--27__PDL1--19__PD1"},{"Channels":["DNA1","CD4","FOXP3"],"Colors":["0000ff","ffffff","ff0000"],"Name":"Helper and Regulatory T Cells","Path":"Helper-and-Regulatory-T-Cells_0__DNA1--17__CD4--26__FOXP3"},{"Channels":["DNA1","CD8a","PD1"],"Colors":["0000ff","ffffff","00ff00"],"Name":"CD8 Cytotoxic T Cells","Path":"CD8-Cytotoxic-T-Cells_0__DNA1--23__CD8a--19__PD1"},{"Channels":["DNA1","CD8a","FOXP3"],"Colors":["0000ff","ffffff","ff0000"],"Name":"FOXP3 CD8 T Cells","Path":"FOXP3-CD8-T-Cells_0__DNA1--23__CD8a--26__FOXP3"},{"Channels":["DNA1","PanCK","Na_K_ATPase"],"Colors":["0000ff","ffffff","00ff00"],"Name":"NaK ATPase","Path":"NaK-ATPase_0__DNA1--14__PanCK--10__Na-K-ATPase"},{"Channels":["DNA1","PanCK","Ecadherin"],"Colors":["0000ff","ffffff","00ff00"],"Name":"E-Cadherin","Path":"E-Cadherin_0__DNA1--14__PanCK--29__Ecadherin"},{"Channels":["DNA1","ASMA","Desmin","Collagen","Vimentin"],"Colors":["0000ff","ff0000","00ff00","ffffff","00ffff"],"Name":"Stroma","Path":"Stroma_0__DNA1--15__ASMA--34__Desmin--39__Collagen--30__Vimentin"},{"Channels":["DNA1","CD4","CD68","CD163","PDL1"],"Colors":["0000ff","ffffff","00ffff","00ff00","ff0000"],"Name":"PDL1-Positive Immune Cells","Path":"PDL1-Positive-Immune-Cells_0__DNA1--17__CD4--22__CD68--25__CD163--27__PDL1"},{"Channels":["DNA1","PDL1","CD8a","PD1"],"Colors":["0000ff","ff0000","ffffff","00ff00"],"Name":"PDL1-CD8 Interaction","Path":"PDL1-CD8-Interaction_0__DNA1--27__PDL1--23__CD8a--19__PD1"},{"Channels":["DNA1","PanCK","Ecadherin","PCNA"],"Colors":["0000ff","ffffff","00ff00","ff0000"],"Name":"Tumor Budding Epithelial","Path":"Tumor-Budding-Epithelial_0__DNA1--14__PanCK--29__Ecadherin--37__PCNA"},{"Channels":["DNA1","PanCK","PDL1","FOXP3","CD8a","PD1","CD68"],"Colors":["0000ff","ffffff","ff0000","ffff00","00ff00","6c00ff","654321"],"Name":"Tumor Budding Immune Modulation","Path":"Tumor-Budding-Immune-Modulation_0__DNA1--14__PanCK--27__PDL1--26__FOXP3--23__CD8a--19__PD1--22__CD68"},{"Channels":["DNA1","LaminABC"],"Colors":["0000ff","00ff00"],"Name":"Nuclear Lamina","Path":"Nuclear-Lamina_0__DNA1--33__LaminABC"},{"Channels":["DNA1","DNA10"],"Colors":["0000ff","ff0000"],"Name":"DAPI Cycle Correlation","Path":"DAPI-Cycle-Correlation_0__DNA1--36__DNA10"},{"Channels":["DNA1","PanCK","Ecadherin","PCNA"],"Colors":["0000ff","ffffff","00ff00","ff0000"],"Name":"Transitions","Path":"Transitions_0__DNA1--14__PanCK--29__Ecadherin--37__PCNA"}],"Header":"Cancer tissue atlases provide valuable resources for the analysis of cell differentiation, microenvironment composition, tissue architecture, and spatial relationships in 2D, 3D, and hyper-dimensional reference frames. Highly multiplexed immunofluorescence methods such as t-CyCIF (Lin, Izar, Wang, Yapp, Mei, P. Shah, et al., 2018) provides a highly customizable and readily applicable methodology to generate large-scale datasets amenable to image-based computational and visual analysis from macroscopic to sub-cellular resolution. However, few such data sets have been generated or rigorously annotated to date. \n\nHere, we present a virtual atlas of a representative 1.6cm x 1.6cm x 5um formalin-fixed paraffin-embedded (FFPE) section of a primary colorectal adenocarcinoma subjected to standard hematoxylin and eosin (H&E) histochemical staining as well as 24-plex cyclic immunofluorescence (CyCIF) encompassing >10^7 cells. The sample was systematically analyzed via histopathologic examination by a pathologist, single cell segmentation and classification of cell-states by marker expression, and machine learning-based approaches to predict cell-state transitions and spatial relationships at both single cell and tissue-level resolutions. These analyses revealed spatially-organized molecular and morphologic transitions and spatial interactions occurring between diverse cell populations in distinct tumor microenvironments. Furthermore, they demonstrate the advantages of whole-slide 2D and 3D imaging to understand tumor biology at different levels of tissue organization. \n","Images":[{"Description":"CRC01-096 and CRC01-097","Height":27120,"MaxLevel":5,"Name":"i0","Path":"https://s3.amazonaws.com/www.cycif.org/lin-wang-coy-2021/reg_96_97","Width":26139},{"Name":"WORKAROUND-decouple-storyname-image-description"}],"Layout":{"Grid":[["i0"]]},"Masks":[],"Rotation":0,"Stories":[{"Description":"","Name":"","Waypoints":[{"ActiveMasks":[],"Arrows":[],"Description":"The tumor is derived from a 69-year old male with a history of cutaneous melanoma and gastric MALT lymphoma associated with helicobacter pylori infection. He presented to medical attention following several syncopal episodes and falls thought to be secondary to anemia and orthostatic hypotension. Colonoscopy revealed a mass in the right ascending colon. He underwent hemicolectomy revealing a 6.2cm exophytic mass involving the ascending colon, cecum, and terminal ileum. \n\nHe underwent right hemicolectomy and the specimen was examined by pathology. \n\nPATHOLOGIC DIAGNOSIS:\n\nINVASIVE ADENOCARCINOMA, high grade (poorly differentiated, grade III), with mucinous and signet ring cell features (6.2cm)\n\nThe tumor resection margins were negative for involvement by adenoma, dysplasia, and adenocarcinoma.\n \nLymphovascular invasion (LVI) was identified at the time of pathologic evaluation of the whole specimen, but is not identified in this tissue section. \n\nPerineural invasion (PNI) was not identified. \n\nNo tumor deposits were present.\n\nThe tumor was associated with a villous adenoma. \n\n3/25 local lymph nodes were positive for involvement by metastatic adenocarcinoma. \n\nMolecular testing was notable for the presence of a BRAF V600E mutation, and microsatellite instability (MSI-H). \n\nAJCC Classification (8th Ediiton) - pT3N1b\n","Group":"Histology","Masks":[],"Name":"Clinical History","Overlays":[],"Pan":[0.4819137168141593,0.5],"Zoom":0.5183593749999998},{"ActiveMasks":[],"Arrows":[{"Angle":300,"HideArrow":false,"Point":[0.8076275240652687,0.3062583349205465],"Text":"Mucosa"},{"Angle":300,"HideArrow":false,"Point":[0.7657619961409615,0.2610435647622948],"Text":"Muscularis Mucosa"},{"Angle":300,"HideArrow":false,"Point":[0.7406426793863773,0.21415417348707078],"Text":"Submucosa"},{"Angle":315,"HideArrow":false,"Point":[0.6410027229265264,0.15554243439304083],"Text":"Muscularis Propria"},{"Angle":325,"HideArrow":false,"Point":[0.5195926919460357,0.09944262697446926],"Text":"Subserosa"},{"Angle":180,"HideArrow":false,"Point":[0.14698949341970224,0.24680928526803037],"Text":"Serosa"},{"Angle":180,"HideArrow":false,"Point":[0.08670313320870005,0.567499229168223],"Text":"Invasive Adenocarcinoma, involving Muscularis Propria"},{"Angle":180,"HideArrow":false,"Point":[0.19136695301946782,0.8630698563138315],"Text":"Invasive Adenocarcinoma, involving Submucosa"},{"Angle":30,"HideArrow":false,"Point":[0.398182660965545,0.9191696637324027],"Text":"Invasive Adenocarcinoma, Superficial/luminal Surface"},{"Angle":30,"HideArrow":false,"Point":[0.2650502821662484,0.96103519165671],"Text":"Ulcer Bed"},{"Angle":230,"HideArrow":false,"Point":[0.409905008784351,0.6403452477565172],"Text":"Invasive Adenocarcinoma, with Mucinous Features"},{"Angle":360,"HideArrow":false,"Point":[0.8586844814160859,0.5312460002708036],"Text":"Invasive Adenocarcinoma, with Solid Architecture"},{"Angle":360,"HideArrow":false,"Point":[0.7434033298612541,0.3707871001336729],"Text":"Invasive Adenocarcinoma, with Signet Ring Cell Features"},{"Angle":180,"HideArrow":false,"Point":[0.7188671388208674,0.5004783956328586],"Text":"Invasive Adenocarcinoma, Tumor Budding Region"},{"Angle":60,"HideArrow":true,"Point":[0.8443854575828106,0.8342125094197439],"Text":"Bowel Lumen"},{"Angle":86,"HideArrow":true,"Point":[0.08699867373352957,0.09540227057748238],"Text":"Peritoneal Cavity"}],"Description":"The colorectum is typically conceptually organized into anatomic layers, which are described proceeding from the luminal surface to the serosal surface. \n\nMucosa - The mucosa occupies the luminal surface of the colorectum and accomplishes the primary functions of the bowel including nutrient absorption, facilitation of waste excretion, and regulation of the enteral microbiome. Histologically, the mucosa consists of a surface columnar epithelium bounded by basement membrane and underlying stroma which is densely populated by chronic inflammatory cells including lymphocytes, monocytic cells, and eosinophils with only rare neutrophils in the resting state (see immune populations marker set). The epithelium exhibits regular invaginations into the stroma with a circumferential parallel array of tubular epithelial \"crypts\" which are arrayed similar to a rack of test tubes, with openings into the bowl lumen. In this section, crypts can be seen in cross-section and in longitudinal profile. The epithelium consists of a mixture of cell types, including prominent mucinous goblet cells which exhibit a clear cytoplasm in standard H&E sections (resulting from a washing out of the mucinous contents during tissue processing), absorptive cells which exhibit a more eosinophilic cytoplasm, and more rarely interspersed neuroendocrine cells. The mucinous goblet cells are more prominent in the colorectum than in the small bowel. The deep aspect of the crypts is hypothesized to harbor stem-like progenitor cells which intermittently generate highly proliferative progenitor cells that terminal differentiate as they approach the luminal surface to continuously regenerate and replace the colorectal epithelial surface. Indeed, examination of the crypts with proliferation markers such as PCNA or Ki-67 shows a much higher degree of cell cycle activity in the deep aspects of the crypts (see proliferation marker set). \n\nMuscularis mucosa - The muscularis mucosa is a thin band of smooth muscle cells which delimits the deep edge of the mucosal compartment. The muscularis mucosa can be identified in this section as a brightly eosinophilic band of spindled cells at the deep surface of the mucosa on H&E. In tumor resections, invasion of atypical cells through the muscularis mucosa into the submucosa represents an important event, as this behavior is the definitive finding defining transition from an \"in situ\" carcinoma with essentially no metastatic risk (pTis), to an \"invasive\" carcinoma with associated increased risk of local or distant metastatic spread (pT1) [AJCC 8th Edition]. The muscularis mucosa can be highlighted here by observing the aSMA, desmin, or collagen channels in the Tissue Structure or Stroma marker sets, respectively. \n\nSubmucosa - The submucosa is a loose collagenous connective tissue structure interposed between the muscularis mucosa and muscularis propria. The submucosa contains a rich bed of lymphatic and blood vessels, and lymphovascular invasion (LVI) is often identified at this level if present. Involvement of the submucosa by tumor defines frank invasion in the colorectum (pT1). \n\nMuscularis propria - The muscularis propria is the thick smooth muscular wall of the bowel which is arranged into several distinct layer, principally including the longitudinal and circumferential layers in the colorectum (these can be seen here as the two distinct layers of smooth muscle oriented in different directions on H&E).  The muscularis regulates digestion by controlling peristalsis and moevement of the bowel wall. Invasion of an invasive adenocarcinoma into the muscularis propria represents a more advanced stage of invasion (pT2) with a higher risk of recurrence and metastatic spread (pT2). The muscularis mucosa can be highlighted here by observing the aSMA, desmin, or collagen channels in the Tissue Structure or Stroma marker sets, respectively. \n\nSubserosa - The subserosa is the loose fibroadipose connective tissue associated with the bowl wall which lies deep to the muscularis propria. Invasion of tumor into the subserosal soft tissue denotes a further progression of stage with increased risk of recurrence and metastatis (pT3). \n\nSerosa - The serosa is the final layer of the bowel wall. In the peritoneal cavity it is lined by a single layer of cuboidal mesothelial cells. Mesothelial cells are specialized epithelial cells and may sometimes cause confusion with invasive tumor when they undergo hyperplasia and reactive changes and exhibit clusters of cells with frond-like papillary architecture; however, they may be differentiated from other epithelial tumor cells by molecular markers including the absence of e-cadherin expression, and expression of mesothelial cell markers such as WT1 and calretinin (see Tissue Structure and E-Cadherin marker sets). In the retroperitoneum (some aspects of the right colon including the cecum), the serosal surface is bounded by a fibrous adventitial layer. Invasion of tumor through the serosal surface into the peritoneal cavity or adjacent structures represents the highest grade of local invasion, with the greatest risk of recurrence and metastatic spread (pT4). This section of bowel exhibits some hyperplasia of the mesothelial cell layer, giving it a more complex tufted appearance, but these cells may be confirmed as mesothelial and not surface involvement by morphologic features and marker expression (see E-cadherin marker set).\n\nIn this case, the tumor is present on the bottom aspect of the frame, while the serosal surface is present at the superior aspect. The bowel wall has been cut in an oblique plane, accounting for the convoluted appearance of the muscularis and serosa. \n\nThe tumor exhibits an exophytic (extending into the bowel lumen) growth pattern on the right-hand side of the field, with prominent mucinous pools generated by mucinous tumor cells present inferiorly extending as deeply as the muscularia propria. The tumor exhibits a moderately differentiated glandular appearance on the left-hand and bottom aspects of the field, while on the right side of the field in the exophytic region of tumor there are intermixed regions with a more poorly differentiated solid sheet-like architecture, with focal signet ring cell features and tumor budding. \n\nCollections of immune cells are present associated with the tumor, in addition to the normal immune complement of the bowel mucosa (see immune marker panels, and subsequent discussion).","Group":"Histology","Masks":[],"Name":"Tissue Organization [Histology]","Overlays":[],"Pan":[0.4819137168141593,0.5],"Zoom":0.5183593749999998},{"ActiveMasks":[],"Arrows":[{"Angle":300,"HideArrow":false,"Point":[0.8157494364825845,0.31650339110776177],"Text":"Mucosa"},{"Angle":300,"HideArrow":false,"Point":[0.7878669948849958,0.2690278824415975],"Text":"Muscularis Mucosa"},{"Angle":300,"HideArrow":false,"Point":[0.7456665427372943,0.20497362471740754],"Text":"Submucosa"},{"Angle":300,"HideArrow":false,"Point":[0.6416725713733152,0.15448379804069312],"Text":"Muscularis Propria"},{"Angle":300,"HideArrow":false,"Point":[0.5246165552969524,0.08649418069161852],"Text":"Subserosa"},{"Angle":180,"HideArrow":false,"Point":[0.13275521392543776,0.2606547768567361],"Text":"Serosa"},{"Angle":180,"HideArrow":false,"Point":[0.08502851209172757,0.5537134723268861],"Text":"Invasive Adenocarcinoma, involving Muscularis Propria"},{"Angle":180,"HideArrow":false,"Point":[0.1913669530194677,0.8677049317591895],"Text":"Invasive Adenocarcinoma, involving Submucosa"},{"Angle":30,"HideArrow":false,"Point":[0.41827811436921225,0.9095704596834966],"Text":"Invasive Adenocarcinoma, superficial/luminal surface"},{"Angle":30,"HideArrow":false,"Point":[0.27677262998505425,0.9757179938039018],"Text":"Ulcer Bed"},{"Angle":220,"HideArrow":false,"Point":[0.39734535040705876,0.6458176337603617],"Text":"Invasive Adenocarcinoma, with Mucinous Features"},{"Angle":360,"HideArrow":false,"Point":[0.7532023377636694,0.37871556560328223],"Text":"Invasive Adenocarcinoma, with Signet Ring Cell Features"},{"Angle":360,"HideArrow":false,"Point":[0.8564432899763368,0.5337370103231389],"Text":"Invasive Adenocarcinoma, with Solid Architecture"},{"Angle":180,"HideArrow":false,"Point":[0.7187075336454248,0.5040935676535856],"Text":"Invasive Adenocarcinoma, Tumor Budding Region"},{"Angle":60,"HideArrow":true,"Point":[0.8444691886386589,0.9102403081302853],"Text":"Bowel Lumen"},{"Angle":60,"HideArrow":true,"Point":[0.08916907985017969,0.1001740538275813],"Text":"Peritoneal Cavity"}],"Description":"Multiplexed immunofluorescence (CyCIF) effectively highlights the principle anatomic compartments of the tissue specimen. \n\nStroma  - Alpha smooth muscle actin (ASMA) [Red] - ASMA highlights smooth muscle cells and myofibroblasts. The muscularis propria is comprised of thick bundles of smooth muscle cells which stain intensely for ASMA in the center of the specimen. The muscularis mucosa is identifiable as a prominent band of ASMA-positive smooth muscle cells bounding the mucosal compartment on the upper right of the specimen. The smooth muscle layer (tunica media) of large veins and arteries and arterioles are also highlighted by ASMA, most prominently seen here in the submucosa and subserosa. Myofibroblasts are visible as spindled cells with less intense ASMA staining, most prominently in the stroma of the mucosa, submucosa, and subserosa. Note that not all stromal fibroblastic cells express ASMA. Smooth muscle cells are also prominently stained by desmin and vimentin, and associated with collagen bundles (see also Stroma marker set). \n\nEpithelial - Pan-Cytokeratin (PanCK) [White] - Pan-cytokeratin highlights both high and low molecular weight cytokeratin intermediate filaments which are specifically expressed by cells with epithelial differentiation in this tissue setting. Accordingly, PanCK intensely highlights normal mucosal epithelium, tumor epithelial cells, and mesothelial cells. Normal epithelium and tumor cells also express other epithelial adhesion molecules such as e-cadherin, but this protein is not present in serosal mesothelial cells (see also E-cadherin, NaK ATPase marker sets)\n\nVascular  - CD31 [Green] - CD31 (PECAM-1) is a cell adhesion molecule that is most strongly expressed on endothelial cells, though it may also be expressed by platelets, monocytes, neutrophils, and T cells. In this representation, the fluorescence intensity has been optimized to highlight endothelial cells, which line the luminal surface of all vascular structures. \n\nImmune - CD45 [Yellow] - CD45 (PTPRC) encodes a surface membrane protein tyrosine phosphatase that is expressed by nearly all hematolymphoid cell populations, and therefore represents an excellent \"pan-immune\" tissue marker. Prominent aggregates of immune cells are present both in the peritumoral compartment as well as normal structures. A variety of marker sets are available to identify specific cell populations. \n\n","Group":"Tissue Structure","Masks":[],"Name":"Tissue Organization [CyCIF]","Overlays":[],"Pan":[0.4819137168141593,0.5],"Zoom":0.5183593749999998},{"ActiveMasks":[],"Arrows":[{"Angle":60,"HideArrow":true,"Point":[0.10693442398913522,0.6738806518865044],"Text":"ROI4"},{"Angle":60,"HideArrow":true,"Point":[0.24090411334691808,0.8476225927723788],"Text":"ROI3"},{"Angle":60,"HideArrow":true,"Point":[0.38254914949082386,0.8922791558916399],"Text":"ROI2"},{"Angle":60,"HideArrow":true,"Point":[0.7251487196714039,0.2873222773854017],"Text":"ROI1"},{"Angle":60,"HideArrow":true,"Point":[0.8551742560623484,0.5284610322835287],"Text":"ROI5"},{"Angle":60,"HideArrow":true,"Point":[0.44000777081296905,0.6860382276652955],"Text":"ROI6"},{"Angle":60,"HideArrow":true,"Point":[0.733066466283119,0.5179946503024517],"Text":"ROI7"}],"Description":"Contemporary pathologic classification of human colorectal adenocarcinomas is principally based on the light microscopic examination of hematoxylin and eosin (H&E) stained tissue sections (Fletcher 2013). The TNM staging system developed by the American Joint Commission on Cancer (AJCC) is the most widely accepted classification schema with high prognostic significance (Amin et al. 2017). Staging of the primary tumor mass is primarily based on the depth of invasion of tumor into the bowel wall (AJCC 8th Edition):\n\nT0 - No evidence of primary tumor\n\nTis - Carcinoma in situ\n\nT1 - Tumor invades submucosa\n\nT2 - Tumor invades muscularis propria\n\nT3 - Tumor invades through the muscularis propria into the pericolorectal tissue\n\nT4 - Tumor invades the visceral peritoneum or invades or adheres to adhacent organs or structures\n\nPrimary tumors may also be independently classified by a variety of morphologic features including grade (low grade vs. high grade), qualitative degree of differentiation (well, moderately, poorly differentiated), type of differentiation (e.g. serous, mucinous, signet ring cell, sarcomatoid), architecture features (glandular, solid, cribriform, budding architecture), \n\nOn histopathologic review this tumor exhibited highly heterogeneous morphologic features with distinct locoregional relationships between tumor region and differentiation. \n\nWe accordingly defined exemplar regions of interest (ROI) to compare tumor compartments with distinctive morphologies as well as different levels of anatomic invasion corresponding to clinicopathologic (AJCC) tumor stages. \n\nROI1 - Normal colonic mucosa\n\nROI2 - Invasive Adenocarvinoma, superficial/luminal\n\nROI3  - Invasive Adenocarcinoma, submucosal\n\nROI4  - Invasive Adenocarcinoma, muscularis\n\nROI5 - Invasive Adenocarcinoma, solid/poorly differentiated\n\nROI6 - Invasive Adenocarcinoma, mucinous\n\nROI7 - Invasive Adenocarcinoma, tumor budding","Group":"Histology","Masks":[],"Name":"Regions of Interest - Overview","Overlays":[],"Pan":[0.4819137168141593,0.5],"Zoom":0.5183593749999998},{"ActiveMasks":[],"Arrows":[{"Angle":210,"HideArrow":false,"Point":[0.6169832102098471,0.3107033075818557],"Text":"Germinal center"},{"Angle":310,"HideArrow":false,"Point":[0.8044537304389947,0.28201699508549527],"Text":"Colonic crypts in longitudinal profile"},{"Angle":180,"HideArrow":false,"Point":[0.577318704807988,0.40210412437744414],"Text":"Colonic crypts in cross-section"},{"Angle":60,"HideArrow":false,"Point":[0.7072904139190641,0.31061918349828865],"Text":"Mucosal stroma, with prominent immune infiltrate"},{"Angle":310,"HideArrow":false,"Point":[0.7573442436414526,0.25635914959754824],"Text":"Muscularis mucosa"},{"Angle":60,"HideArrow":true,"Point":[0.7943588404109498,0.362355494892018],"Text":"Bowel Lumen"},{"Angle":60,"HideArrow":true,"Point":[0.7373647737942806,0.43091662299915523],"Text":"Adenocarcinoma, poorly differentiated region"},{"Angle":60,"HideArrow":true,"Point":[0.7108656874706631,0.23364564703444765],"Text":"Submucosa"},{"Angle":60,"HideArrow":true,"Point":[0.4511325794574287,0.39516388748316356],"Text":"Muscularis Propria"}],"Description":"Region of Interest #1 (ROI1) consists of a region of normal colonic mucosa adjacent to the primary tumor mass. No cytologic atypia or adenomatous changes are evident.\n\nOne may appreciate the typical mucosal architecture including parallel test tube-like crypts see here in cross-section and longitudinal profile. The colonic epithelium is well-ordered with columnar absorptive cells and prominent mucinous goblet cells, small basal nuclei, and well-spaced glands with smooth contours. A normal mucosal germinal center responsible for B cell maturation and plasma cell differentiation is present in the top left of the field. The surface of the colorectum is normally a relatively flat surface, but the mucosa is curved and distorted here by the adjacent exophytic tumor mass. One may note the dense lymphocytic infiltrate in the mucosal stromal, which is a normal finding. ","Group":"Histology","Masks":[],"Name":"*ROI1 - Normal Mucosa","Overlays":[{"height":0.08782554324398928,"width":0.2465676889349928,"x":0.5745152549510262,"y":0.2528827906843446},{"height":0.16000400694450917,"width":0.07823539771734667,"x":0.5583634309061547,"y":0.326070743387669}],"Pan":[0.6931996299215849,0.35257607017734977],"Zoom":1.8573753599999985},{"ActiveMasks":[],"Arrows":[{"Angle":60,"HideArrow":true,"Point":[0.4044741901119763,0.9413097892289416],"Text":"Bowel Lumen"},{"Angle":180,"HideArrow":false,"Point":[0.3173317655836193,0.9169683298634788],"Text":"Adenocarcinoma, with glandular architecture"},{"Angle":180,"HideArrow":false,"Point":[0.34873224816506637,0.9337639368256482],"Text":"Peritumoral stroma"},{"Angle":360,"HideArrow":false,"Point":[0.374534195092457,0.8904361391551243],"Text":"Glandular lumen with mucin"},{"Angle":360,"HideArrow":false,"Point":[0.4406212572696886,0.8842290670169313],"Text":"Adenocarcinoma with cribriform architecture"}],"Description":"Region of Interest #2 (ROI2) consists of a region of invasive adenocarcinoma with a predominantly moderately differentiated glandular appearance and typical cytologic features. ROI2 is defined at the most superifical/luminal region of the tumor, though like ROI3 the tumor in this region predominantly involves the submucosal compartment (pT1 anatomic level). One may appreciate a prominent ulcer bed adjacent to this region, often present in the center of the superficial aspect of the tumor mass in colorectal adenocarcinomas. \n\nCompared to normal colonic mucosa, the adenocarcinomatous epithelium has a disordered arrangement of epithelial glandular structures which exhibit jagged erratic shapes. Cytologically, the tumor cells have an eosinophilic to amphophilic cytoplasm without prominent mucin vacuoles, with cuboidal to pseudo-stratified columnar arrangement. Glands are densely arranged with back to back glandular structures with minimal intervening stroma and focal regions of more disordered cribriform growth (multiple small lumens in a single glandular structure). The tumor cell nuclei are larger and more irregular than those of the normal mucosa, with darker hematoxylin staining of chromatin. \n\nThough less prominent than other tumor regions (ROI6), scattered mucinous cells and intraluminal mucin are present focally. The peritumoral stromal has a prominent lymphocytic inflammatory infiltrate in this region. \n\n[Regions of interest 2-3 (ROI2-4) each exhibit relatively similar morphologic and architectural features, and were selected to compare the molecular and spatial relationships of tumor and tumor-associated cells at different anatomic levels of invasion corresponding to defined AJCC staging criteria.]","Group":"Histology","Masks":[],"Name":"*ROI2 - Adenocarcinoma, Superficial","Overlays":[{"height":0.11708241954787624,"width":0.147630951051532,"x":0.30589127968185204,"y":0.8354244409891782}],"Pan":[0.3806195599338227,0.9002335765497231],"Zoom":3.209544622079997},{"ActiveMasks":[],"Arrows":[{"Angle":360,"HideArrow":false,"Point":[0.24982434068431564,0.867594646418917],"Text":"Glandular lumen with mucin"},{"Angle":180,"HideArrow":false,"Point":[0.1924190656807658,0.8815909855540581],"Text":"Adenocarcinoma, with glandular architecture"},{"Angle":310,"HideArrow":false,"Point":[0.24576743079007185,0.8449773737585078],"Text":"Peritumoral stroma"},{"Angle":60,"HideArrow":true,"Point":[0.33238245703217706,0.9189145565811011],"Text":"ROI1"},{"Angle":60,"HideArrow":true,"Point":[0.2705145811449591,0.9385905695681835],"Text":"Ulcer Bed"}],"Description":"Region of Interest #3 (ROI3) consists of a region of invasive adenocarcinoma with a predominantly moderately differentiated glandular appearance and typical cytologic features. ROI3 is a well-defined region of invasive adenocarcinoma involving the submucosal compartment (pT1 anatomic level). \n\nCompared to ROI2,  the carcinomatous glands exhibit a more prominent pseudo-stratified columnar cytomorphology with disorganized and crowded back-to-back glandular structures. Regions of cribriform growth are not identified in this region, suggesting that this region of tumor is possibly somewhat more well-differentiated than the superificial tumor in ROI2. The peritumoral stroma is less prominent in this region, and significantly fewer peritumoral immune cells are present. While the tumor in this region exhibits a predominantly typical morphology, some scattered mucinous cells and intraluminal mucin are present.\n\n[Regions of interest 2-3 (ROI2-4) each exhibit relatively similar morphologic and architectural features, and were selected to compare the molecular and spatial relationships of tumor and tumor-associated cells at different anatomic levels of invasion corresponding to defined AJCC staging criteria.]\n","Group":"Histology","Masks":[],"Name":"*ROI3 - Adenocarcinoma, Submucosa","Overlays":[{"height":0.08296380733728581,"width":0.10537823450298284,"x":0.17963979951389786,"y":0.837877781443581}],"Pan":[0.23166839959811703,0.8834469297807171],"Zoom":3.8514535464959962},{"ActiveMasks":[],"Arrows":[{"Angle":180,"HideArrow":false,"Point":[0.07897746771222469,0.5905234399709136],"Text":"Adenocarcinoma, with glandular architecture"},{"Angle":180,"HideArrow":false,"Point":[0.09247237278443729,0.6082245492214782],"Text":"Glandular lumen with mucin"},{"Angle":360,"HideArrow":false,"Point":[0.14329733993952368,0.6101523928032229],"Text":"Extracellular mucin pool"},{"Angle":360,"HideArrow":false,"Point":[0.17256551068055617,0.6318844477247081],"Text":"Peritumoral lymphohistiocytic inflammation"},{"Angle":360,"HideArrow":false,"Point":[0.10421469278233655,0.5230489146098507],"Text":"Deepest invasive front of tumor"},{"Angle":60,"HideArrow":true,"Point":[0.24319468917538314,0.6092761002660663],"Text":"Muscularis Propria"}],"Description":"Region of Interest #4 (ROI4) consists of a region of invasive adenocarcinoma with a predominantly moderately differentiated glandular appearance and typical cytologic features. ROI4 is a well-defined region of invasive adenocarcinoma involving the muscularis propria, making it a higher stage of invasion than ROI2-3 (pT2 anatomic level). ROI4 also represents the deepest extent of invasion in this tissue section at its superior aspect in this field. \n\nWhile the adenocarcinomatous glands in this region maintain a glandular architecture with predominantly typical cytomorphology and columnar architecture, the glands exhibit somewhat more prominent intraluminal and extracellular mucin, though not to the extent of the mucinous region in ROI6. The tumor region exhibits relatively small amounts of peritumoral stroma and intratumoral immune infiltration, though the deep edges of the mass exhibit collections of lymphohistiocytic inflammation, most prominently around regions of extracellular mucin. \n\n[Regions of interest 2-3 (ROI2-4) each exhibit relatively similar morphologic and architectural features, and were selected to compare the molecular and spatial relationships of tumor and tumor-associated cells at different anatomic levels of invasion corresponding to defined AJCC staging criteria.]\n","Group":"Histology","Masks":[],"Name":"*ROI4 - Adenocarcinoma, Muscularis","Overlays":[{"height":0.2181968417520087,"width":0.10515510445879939,"x":0.06390523607313017,"y":0.5144612477457154}],"Pan":[0.10999822352757051,0.6247864781737391],"Zoom":2.228850431999998},{"ActiveMasks":[],"Arrows":[{"Angle":60,"HideArrow":true,"Point":[0.90190277447454,0.5254714654665518],"Text":"Bowel Lumen"},{"Angle":60,"HideArrow":true,"Point":[0.8562390606764834,0.4951855061866067],"Text":"Colonic Mucosa"},{"Angle":60,"HideArrow":false,"Point":[0.8604650084829873,0.5325147118107251],"Text":"Adenocarcinoma, with solid architecture"},{"Angle":180,"HideArrow":false,"Point":[0.8565912229936921,0.5320451620544469],"Text":"Intracellular mucin vacuole"},{"Angle":360,"HideArrow":false,"Point":[0.8885793001401456,0.5041656452754276],"Text":"Discohesive and fragmented adenocarcinoma, with focal signet ring cell features"}],"Description":"Region of Interest #5 (ROI5) consists of a region of adenocarcinoma with near-complete loss of glandular morphology and a solid sheet-like architecture, suggestive of a more poorly differentiated state. \n\nHowever, scattered tumor cells still exhibit intracellular mucin vacuoles, exposing the underlying cytomorphologic differentiation despite the loss of glandular architecture. At the top-right region of the field some cells have become increasingly discohesive with more prominent mucin vacuoles, with focal signet ring cell morphology (cells with a promeint cytoplasmic mucin vacuole and nucleus 'pushed' to the periphery of the cell).\n\nThis region of tumor is present in both the mucosa and submucosa, notice the mucosal glands and stroma adjacent to the solid tumor to the left, and a region of more glandular architecture inferiorly. ","Group":"Histology","Masks":[],"Name":"*ROI5 - Adenocarcinoma, Solid","Overlays":[{"height":0.059163269291055476,"width":0.057989394900359925,"x":0.8401569815239543,"y":0.4852075738656945}],"Pan":[0.8728493833048253,0.5164619795179635],"Zoom":6.6553117283450804},{"ActiveMasks":[],"Arrows":[{"Angle":60,"HideArrow":true,"Point":[0.41393839026196283,0.6093008987564413],"Text":"Muscularis propria"},{"Angle":60,"HideArrow":true,"Point":[0.4483114908074552,0.7408877902720046],"Text":"Submucosa"},{"Angle":310,"HideArrow":false,"Point":[0.46630505946728557,0.6327460473467731],"Text":"Peritumoral inflammation"},{"Angle":200,"HideArrow":false,"Point":[0.4153258648256596,0.6370363756086921],"Text":"Invasive Front"},{"Angle":310,"HideArrow":false,"Point":[0.4357680171324502,0.6654282538125679],"Text":"Extracellular mucin"},{"Angle":310,"HideArrow":false,"Point":[0.48498060601916837,0.6660591844393207],"Text":"Adenocarcinoma, with cribriform architecture"},{"Angle":180,"HideArrow":false,"Point":[0.3974074350258802,0.6909178511333809],"Text":"Carcinomatous glandular lumen in cribfriform structure"}],"Description":"Region of interest #6 (ROI) comprises a region of \"floating\" tumor with prominent extracellular mucin surrounding the tumor epithelium. \n\nIn this tumor, prominent mucinous regions are present at the deep invasive front broadly across the tumor. The tumor in this region exhibits a prominent cribriform architecture with multiple small lumens within a sheet-like structure of tumor epithelium with no intervening stroma. Minimal peritumoral stromal is present immediately adjacent to the tumor epithelium. There is minimal inflammatory cell infiltration into the mucin pools and tumor epithelium, though prominent lymphohistiocytic inflammation is present surrounding the extracellular mucin pools. \n\nMacrophage populations are particularly prominent in this region (see Macrophage marker set), likely at least in part resulting from a phagocytic response of macrophages to extracellular mucinous material. ","Group":"Histology","Masks":[],"Name":"*ROI6 - Adenocarcinoma, Mucinous","Overlays":[{"height":0.09176255035492664,"width":0.1553603575316086,"x":0.3719941221954369,"y":0.6374656419346862}],"Pan":[0.45007870889792206,0.6759309098924081],"Zoom":3.0956255999999978},{"ActiveMasks":[],"Arrows":[{"Angle":60,"HideArrow":true,"Point":[0.8357035079692312,0.50829264236419],"Text":"Mucosa"},{"Angle":60,"HideArrow":true,"Point":[0.6348151964111408,0.5511959249833802],"Text":"Submucosa"},{"Angle":60,"HideArrow":false,"Point":[0.7463637312210352,0.5392082430750771],"Text":"Prominent peritumoral lymphohistiocytic inflammation"},{"Angle":180,"HideArrow":false,"Point":[0.7069936601116606,0.5212898132752977],"Text":"Adenocaricnoma, with glandular archiecture"},{"Angle":310,"HideArrow":false,"Point":[0.7351258332580947,0.5150750520646571],"Text":"Tumor Budding Cell(s)"},{"Angle":310,"HideArrow":false,"Point":[0.7237979453878827,0.5022798212060756],"Text":"Tumor Budding Cell(s)"},{"Angle":310,"HideArrow":false,"Point":[0.7149351937381312,0.49241927632423305],"Text":"Tumor Budding Cell(s)"}],"Description":"Region of Interest #7 (ROI) comprises a region of tumor with clusters of tumor budding cells extending into the peritumoral stroma. \n\nTumor budding is defined as clusters of <=4 tumor cells, typically located at the invasive front of the tumor. A prominent peritumoral lymphohistiocytic inflammatory proliferation is present adjacent to the region of tumor budding. \n\nTumor budding cells may be difficult to appreciate on standard H&E sections, but are highlighted by PanCK (see also, Tumor Budding marker sets, and subsequent discussion).  ","Group":"Histology","Masks":[],"Name":"*ROI7 - Tumor Budding Cells","Overlays":[{"height":0.058676548288010055,"width":0.07142134694841651,"x":0.6875609968076746,"y":0.48949090968695674}],"Pan":[0.7241337185223892,0.5217584342633428],"Zoom":3.8514535464959962},{"ActiveMasks":[],"Arrows":[{"Angle":310,"HideArrow":false,"Point":[0.7270804932567821,0.5024769774214954],"Text":"Tumor Budding Cell(s)"},{"Angle":310,"HideArrow":false,"Point":[0.7344172081986293,0.5136776955660488],"Text":"Tumor Budding Cell(s)"},{"Angle":310,"HideArrow":false,"Point":[0.7533948441815407,0.522824133526885],"Text":"Tumor Budding Cell(s)"},{"Angle":310,"HideArrow":false,"Point":[0.7546665414381275,0.5021835088238216],"Text":"Tumor Budding Cell(s)"},{"Angle":30,"HideArrow":false,"Point":[0.7402661721485223,0.527883887596454],"Text":"Poorly Differentiated Cluster"},{"Angle":330,"HideArrow":false,"Point":[0.7131847513916093,0.49370733446411363],"Text":"Poorly Differentiated Cluster"},{"Angle":60,"HideArrow":true,"Point":[0.7038411615732647,0.5209886181048415],"Text":"Main Tumor Mass"},{"Angle":60,"HideArrow":true,"Point":[0.7342702856340766,0.4917586713469188],"Text":"Main Tumor Mass"},{"Angle":30,"HideArrow":false,"Point":[0.7268253932119733,0.5269845046192871],"Text":"Poorly Differentiated Cluster"},{"Angle":60,"HideArrow":true,"Point":[0.6824558330050752,0.5400755235091603],"Text":"Submucosa"}],"Description":"Tumor budding is a phenomenon in which single tumor cells or small clusters of <=4 cells are present in the peri-tumoral stroma, most characteristically at the deep invasive front of a tumor, resulting in an appearance of cells sprouting or budding from the tumor mass [1]. Tumor budding was first conceptually described in 1949 by Imai et al. in their classification of gastric carcinomas [2], and subsequently shown to correlate with clinical outcome in a variety of solid cancers including for colorectal carcinomas, though the concept was only minimally explored until the 1980s [3]. Since that period, a number of studies have explored the possibility of using tumor budding as a novel tumor classifier, and systematic meta-analyses show that tumor budding significantly correlates with poor outcomes in colorectal carcinomas including risk of lymph node metastases, local recurrence, distant metastases, and cancer-related death, independently of tumor TNM stage [4]. The related concept of poorly differentiated clusters (PDCs) refers to similar collections of cells at the invasive front that have lost glandular differentiation, but which form larger clusters and do not meet the definition of tumor budding (>=5 cells) [5]. \n\nPathologic examination of the CRC01 tissue specimen revealed a region of prominent tumor budding near the invasive front of the tumor. Tumor budding cells were annotated by a pathologists using the ITBCC consensus criteria of <=4 cells and were segmented and analyzed for marker expression and neighborhood spatial correlations at a single cell level. Scattered poorly differentiated clusters were also present in the region of tumor budding, and were noted but not included in the training cohort of tumor budding cells. \n\nSingle cell analysis of marker expression in tumor budding cells relative to other tumor cell populations in the main mass (ROI2-6) and normal mucosa (ROI1) showed that these cells exhibit a unique molecular phenotype characterized by reduced expression of characteristic epithelial markers such as E-cadherin and NaK ATPase and reduced proliferative cell cycle activity as assessed by PCNA and Ki-67/MIB-1 staining (highlighted in this field). On the other hand, tumor budding cells at the invasive tumor margin exhibited higher expression of PD-L1 compared to the main tumor mass, indicating that they may suppress local anti-tumor cytotoxic immunity. \n\nInterestingly, these markers exhibited a continuous graded expression profile correlating with tumor cluster size with tumor budding cells representing just one end of a continuous spectrum of molecular differentiation. Larger clusters, including those that would be classified as poorly differentiated clusters (PDCs) also exhibited reduced expression of these markers relative to the main tumor mass, though at an intermediate level to tumor budding cells and the main mass. \n\n3D reconstruction and analysis of tumor budding regions showed that tumor buds in fact rarely represent individual cells or clusters which have completely separated from the main tumor mass, but rather, represent the tips of a contiguous complex network of tendril-like structures connected to the main tumor mass. Poorly differentiated clusters in the region of tumor budding rather than representing distinct tumor nests, instead represent the more proximal aspects of this network. These findings suggest that while tumor budding cells exhibit a migratory molecular program, they have not undergone definitive epithelial-to-mesenchymal differentiation and remain an intrinsic yet specialized epithelial niche of the overall tumor mass. \n\nHere, we highlight tumor budding cells and poorly differentiated clusters (arrows) at the invasive front of the tumor. Tumor budding cells exhibit significantly reduced expression of E-cadherin (green) and PCNA (red) compared to adjacent PDCs or the main tumor mass, while many PDCs exhibit an intermediate expression pattern, though this is more difficult to appreciate qualitatively. \n\n[1] Lugli, A. et al. Recommendations for reporting tumor budding in colorectal cancer based on the International Tumor Budding Consensus Conference (ITBCC) 2016. Mod. Pathol. 30, 1299–1311.\n\n[2] Imai T. Histological comparison of cancer of the stomach in autopsy--and operation--cases. Jpn J Cancer Res 1949;40:199-201.\n\n[3] Hayashida K, Isomoto H, Shirouzu K, et aL A study of invasive colorectal carcinoma with reference mainly to vessel invasion and budding. Nippon Dai- cho-komonbyo Gakkai Zasshi 1987;40:119-26.\n\n[4] Rogers, A. C. et al. Systematic review and meta-analysis of the impact of tumour budding in colorectal cancer. Br. J. Cancer 115, 831–840 (2016).\n\n[5] Ueno H, Kajiwara Y, Shimazaki H, Shinto E, Hashiguchi Y, Nakanishi K, et al. New criteria for histologic grading of colorectal cancer. Am J Surg Pathol. 2012;36:193–201. ","Group":"Tumor Budding Epithelial","Masks":[],"Name":"Tumor Budding [Tumor Markers]","Overlays":[{"height":0.04733061543284439,"width":0.07508100602888712,"x":0.6896834716436998,"y":0.487993036181762}],"Pan":[0.7188181388555379,0.5138074662498382],"Zoom":6.655311728345082},{"ActiveMasks":[],"Arrows":[{"Angle":310,"HideArrow":false,"Point":[0.7343145940928659,0.5137528147452811],"Text":"Tumor Budding Cell(s)"},{"Angle":310,"HideArrow":false,"Point":[0.7227519313445149,0.5004293404108867],"Text":"Tumor Budding Cell(s)"},{"Angle":310,"HideArrow":false,"Point":[0.7118348995110463,0.4901579394923007],"Text":"Tumor Budding Cell(s)"},{"Angle":60,"HideArrow":true,"Point":[0.7006830927994386,0.5197395741378285],"Text":"Main Tumor Mass"},{"Angle":60,"HideArrow":true,"Point":[0.7404187409244828,0.482175593635571],"Text":"Main Tumor Mass"},{"Angle":120,"HideArrow":false,"Point":[0.727506122626832,0.5089399297434295],"Text":"FOXP3+ Treg"},{"Angle":120,"HideArrow":false,"Point":[0.7351949998858878,0.5283675509094408],"Text":"Macrophages (PD-L1 Positive)"},{"Angle":120,"HideArrow":false,"Point":[0.7118935932305812,0.5007228090085607],"Text":"FOXP3+ Treg"},{"Angle":310,"HideArrow":false,"Point":[0.7319081515919402,0.5066508746815732],"Text":"Sparse CD8+ Cytotoxic T Cells"},{"Angle":310,"HideArrow":true,"Point":[0.7525096471486471,0.5412214754875571],"Text":"Lymphocyte Aggregate with abundant CD4 and CD8 T cells"}],"Description":"Immune populations were segmented into broad functional categories by lineage-specific marker expression:\n\nCytotoxic T Cell - CD45/CD3/CD8+ \n\nActivated Cytotoxic T Cell - CD45/CD3/CD8/PD-1+\n\nHelper T Cell - CD45/CD3/CD4+\n\nT Regulatory Cells - CD45/CD3/FOXP3+, CD4 or CD8\n\nMacrophages - CD45, CD68 or CD163\n\nNearest neighbor analysis and spatial correlation showed that the tumor budding region exhibited a distinctive immune microenvironment. Tumor budding cells at the tumor invasive front exhibited higher levels of PD-L1 staining. There were increased numbers of FOXP3+ Tregs in the budding microenvironment, and an increased number of PD-L1 positive macrophages. There were a reduced number of CD8-positive cytotoxic T cells in the budding niche, and those that were present were negative for PD1 suggesting that they were not activated. Collectively, these results suggest that the tumor budding network exists in an immunosuppressive environment that may facilitate stromal invasion and metastasis. ","Group":"Tumor Budding Immune Modulation","Masks":[],"Name":"Tumor Budding [Immune Markers]","Overlays":[{"height":0.0474658457626525,"width":0.07992112491660286,"x":0.6836850235961687,"y":0.486997982182443}],"Pan":[0.7209330131695404,0.5123007200478422],"Zoom":7.986374074014096},{"ActiveMasks":[],"Arrows":[{"Angle":360,"HideArrow":false,"Point":[0.8591319589110643,0.5010370511561657],"Text":"Superficial mucosal epithelium with typical mucosal molecular pattern (ROI1-like)"},{"Angle":60,"HideArrow":true,"Point":[0.8259582686300082,0.4833585028322908],"Text":"Adenocarcinoma, with glandular architecture"},{"Angle":60,"HideArrow":true,"Point":[0.8531451995185171,0.5282239820446746],"Text":"Adenocarcinoma, with solid architecture"},{"Angle":60,"HideArrow":true,"Point":[0.8927282439727708,0.47011719970524507],"Text":"Bowel Lumen"},{"Angle":180,"HideArrow":false,"Point":[0.7862694742669845,0.528357505588102],"Text":"Atypical deep mucosal epithelium, with molecular pattern resembling glandular adenocarcinoma (ROI2-4-like)"}],"Description":"Machine-learning based analysis of tumor molecular expression patterns derived from training on exemplar morphologic regions (ROI1-7) revealed regions with atypical expression patterns, as well as those with transitional or indeterminate morphologic and molecular patterns. \n\nHistologic review of the H&E stained sections showed a region of mucosal epithelium without overt adenomatous or high-grade dysplastic changes surrounded by tumor in the exophytic region of the tumor mass, though definitive histologic analysis is challenging in this setting given reactive changes and tissue distortion by tumor mass effect. Molecular clustering analysis showed that while superifical regions of epithelium in this region exhibited an expression pattern typical for normal mucosa (ROI1), the deeper crypt epithelium exhibited a molecular pattern more closely resembling regions of moderately differentiated invasive adenocarcinoma with glandular morphology (ROI2-4). \n\nWhile the molecular expression pattern is informed by the full array of high-plex tissue markers and not easily amenable to reduction to specific markers, the deep epithelium noticably exhibited markedly elevated proliferative cell cycle activity as assessed by PCNA and Ki-67, and reduced expression of E-cadherin, PanCK, and NaK ATPase relative to more superficial regions of epithelium, which may also be noted on surrounding regions of invasive adenocarcinoma. These findings suggest that while the deeper regions of epithelium exhibit no definitive carcinomatous changes, they have acquired an atypical molecular pattern readily identifiable by molecular profiling, and may represent regions of residual adenomatous or pre-neoplastic epithelium associated with the frankly invasive adenocarcinomatous mass. ","Group":"Histology","Masks":[],"Name":"Transitions - Atypical Mucosa/Glandular Adenocarcinoma [Histology]","Overlays":[{"height":0.0842372262763123,"width":0.10092972011200296,"x":0.7774969645714493,"y":0.4639147680602747}],"Pan":[0.815741853917366,0.5066025043697544],"Zoom":4.621744255795196},{"ActiveMasks":[],"Arrows":[{"Angle":180,"HideArrow":false,"Point":[0.7857981699265565,0.5286510518271874],"Text":"Atypical deep mucosal epithelium, with molecular pattern resembling glandular adenocarcinoma (ROI2-4-like)"},{"Angle":360,"HideArrow":false,"Point":[0.859836775496506,0.49932297405004994],"Text":"Superficial mucosal epithelium with typical mucosal molecular pattern (ROI1-like)"},{"Angle":60,"HideArrow":true,"Point":[0.8931372442117571,0.4692342256677417],"Text":"Bowel Lumen"},{"Angle":60,"HideArrow":true,"Point":[0.81081578094106,0.48444763777115596],"Text":"Adenocarcinoma, with glandular architecture"},{"Angle":166,"HideArrow":true,"Point":[0.8578928395055141,0.5300878740813988],"Text":"Adenocarcinoma, with solid architecture"}],"Description":"Machine-learning based analysis of tumor molecular expression patterns derived from training on exemplar morphologic regions (ROI1-7) revealed regions with atypical expression patterns, as well as those with transitional or indeterminate morphologic and molecular patterns. \n\nHistologic review of the H&E stained sections showed a region of mucosal epithelium without overt adenomatous or high-grade dysplastic changes surrounded by tumor in the exophytic region of the tumor mass, though definitive histologic analysis is challenging in this setting given reactive changes and tissue distortion by tumor mass effect. Molecular clustering analysis showed that while superifical regions of epithelium in this region exhibited an expression pattern typical for normal mucosa (ROI1), the deeper crypt epithelium exhibited a molecular pattern more closely resembling regions of moderately differentiated invasive adenocarcinoma with glandular morphology (ROI2-4). \n\nWhile the molecular expression pattern is informed by the full array of high-plex tissue markers and not easily amenable to reduction to specific markers, the deep epithelium noticably exhibited markedly elevated proliferative cell cycle activity as assessed by PCNA and Ki-67, and reduced expression of E-cadherin, PanCK, and NaK ATPase relative to more superficial regions of epithelium, which may also be noted on surrounding regions of invasive adenocarcinoma. These findings suggest that while the deeper regions of epithelium exhibit no definitive carcinomatous changes, they have acquired an atypical molecular pattern readily identifiable by molecular profiling, and may represent regions of residual adenomatous or pre-neoplastic epithelium associated with the frankly invasive adenocarcinomatous mass. ","Group":"Transitions","Masks":[],"Name":"Transitions - Atypical Mucosa/Glandular Adenocarcinoma [CyCIF]","Overlays":[{"height":0.08206790640230693,"width":0.1011691904877049,"x":0.7778479250641331,"y":0.46684932588380273}],"Pan":[0.8040473923401226,0.50686721376622],"Zoom":4.621744255795196},{"ActiveMasks":[],"Arrows":[{"Angle":360,"HideArrow":false,"Point":[0.8225632312027273,0.47947225029561563],"Text":"Adenocarcinoma, with glandular architecture"},{"Angle":360,"HideArrow":false,"Point":[0.8236392827275316,0.456092585347596],"Text":"Adenocarcinoma, with solid and signet ring cell morphology"}],"Description":"Analysis of molecular expression clusters derived from ROI analysis revealed that many regions of tumor epithelium exhibit mixed or transitional expression patterns. \n\nSome regions of tumor (including the field shown) smoothly transition, or rapidly oscillate, between regions of solid architecture (ROI5-like) and glandular architecture (ROI3-like) in adjacent epithelium. While broad morphologic changes are observable on visual review of H&E histology, molecular analysis of high dimensional imaging data illuminates the remarkable degree of heterogeneity in differentiation and architecture in even small regions of tumor. Moreover, analysis of transitional regions in high dimensional space reveals a continuous gradient of expression states between exemplars of each morphologic or expression group, suggesting that many cells lie in an intermediate differentiation state not readily identifiable by morphologic review or analysis of small groups of markers. These results further suggest that some tumors may exhibit highly plastic phenotypes, with rapid shift between different molecular and morphologic states in closely contiguous regions of tumor. \n\nHigh-dimensional molecular clusters are informed by the entire panel of markers and are not easily demonstrable by limited sets of markers, though the transitional changes in morphologic features may be highlighted by PanCK which demonstrates the tissue distribution and organization of epithelial cells. ","Group":"Histology","Masks":[],"Name":"Transitions - Glandular/Solid [Histology]","Overlays":[{"height":0.052775436148353894,"width":0.06515002868360276,"x":0.7776625357586228,"y":0.44929389616815096}],"Pan":[0.7942455808271774,0.47295335647215253],"Zoom":5.546093106954236},{"ActiveMasks":[],"Arrows":[{"Angle":360,"HideArrow":false,"Point":[0.8376035508860171,0.4761285965480299],"Text":"Adenocarcinoma, with glandular architecture"},{"Angle":360,"HideArrow":false,"Point":[0.7984548399563207,0.45576187586946215],"Text":"Adenocarcinoma, with solid architecture"},{"Angle":360,"HideArrow":false,"Point":[0.846348915096699,0.46080953574945305],"Text":"Adenocarcinoma, with solid architecture"},{"Angle":180,"HideArrow":false,"Point":[0.7818445173279788,0.48962815204102866],"Text":"Adenocarcinoma, with glandular architecture"}],"Description":"Analysis of molecular expression clusters derived from ROI analysis revealed that many regions of tumor epithelium exhibit mixed or transitional expression patterns. \n\nSome regions of tumor (including the field shown) smoothly transition, or rapidly oscillate, between regions of solid architecture (ROI5-like) and glandular architecture (ROI3-like) in adjacent epithelium. While broad morphologic changes are observable on visual review of H&E histology, molecular analysis of high dimensional imaging data illuminates the remarkable degree of heterogeneity in differentiation and architecture in even small regions of tumor. Moreover, analysis of transitional regions in high dimensional space reveals a continuous gradient of expression states between exemplars of each morphologic or expression group, suggesting that many cells lie in an intermediate differentiation state not readily identifiable by morphologic review or analysis of small groups of markers. These results further suggest that some tumors may exhibit highly plastic phenotypes, with rapid shift between different molecular and morphologic states in closely contiguous regions of tumor. \n\nHigh-dimensional molecular clusters are informed by the entire panel of markers and are not easily demonstrable by limited sets of markers. However, one many note relatively reduced expression of E-cadherin and PCNA in regions of solid architecture, which is also noted in ROI5.","Group":"Transitions","Masks":[],"Name":"Transitions - Glandular/Solid [CyCIF]","Overlays":[{"height":0.07571489819986271,"width":0.07994084600636664,"x":0.7642950951870804,"y":0.43439736195880324}],"Pan":[0.7841454112157894,0.4723896154548985],"Zoom":4.621744255795196},{"ActiveMasks":[],"Arrows":[{"Angle":360,"HideArrow":false,"Point":[0.7150863589327308,0.4045477391017453],"Text":"Adenocarcinoma, with solid and signet ring cell morphology"},{"Angle":360,"HideArrow":false,"Point":[0.7130320787490135,0.43201639984402107],"Text":"Adenocarcinoma, with mucinous morphology"}],"Description":"Analysis of molecular expression clusters derived from ROI analysis revealed that many regions of tumor epithelium exhibit mixed or transitional expression patterns. \n\nWhile broad morphologic changes are observable on visual review of H&E histology, molecular analysis of high dimensional imaging data illuminates the remarkable degree of heterogeneity in differentiation and architecture in even small regions of tumor. Moreover, analysis of transitional regions in high dimensional space reveals a continuous gradient of expression states between exemplars of each morphologic or expression group, suggesting that many cells lie in an intermediate differentiation state not readily identifiable by morphologic review or analysis of small groups of markers. \n\nThese results further suggest that some tumors may exhibit highly plastic phenotypes, with rapid shift between different molecular and morphologic states in closely contiguous regions of tumor. ","Group":"Histology","Masks":[],"Name":"Transitions - Solid/Mucinous [Histology]","Overlays":[{"height":0.06556088472034621,"width":0.060924080877098774,"x":0.6724747185504825,"y":0.37801817787202596}],"Pan":[0.6864411042934547,0.4095585545021762],"Zoom":5.546093106954236},{"ActiveMasks":[],"Arrows":[{"Angle":360,"HideArrow":false,"Point":[0.7172833335497912,0.39610853342006946],"Text":"Adenocarcinoma, with solid and signet ring cell morphology"},{"Angle":360,"HideArrow":false,"Point":[0.7115990655472066,0.4306979089251591],"Text":"Adenocarcinoma, with mucinous features"},{"Angle":60,"HideArrow":true,"Point":[0.6493140012635662,0.44412245676105055],"Text":"Mucosa"}],"Description":"Analysis of molecular expression clusters derived from ROI analysis revealed that many regions of tumor epithelium exhibit mixed or transitional expression patterns. \n\nWhile broad morphologic changes are observable on visual review of H&E histology, molecular analysis of high dimensional imaging data illuminates the remarkable degree of heterogeneity in differentiation and architecture in even small regions of tumor. Moreover, analysis of transitional regions in high dimensional space reveals a continuous gradient of expression states between exemplars of each morphologic or expression group, suggesting that many cells lie in an intermediate differentiation state not readily identifiable by morphologic review or analysis of small groups of markers. \n\nThese results further suggest that some tumors may exhibit highly plastic phenotypes, with rapid shift between different molecular and morphologic states in closely contiguous regions of tumor. ","Group":"Transitions","Masks":[],"Name":"Transitions - Solid/Mucinous [CyCIF]","Overlays":[{"height":0.06567143671071218,"width":0.0660947332640962,"x":0.6738047304236383,"y":0.3765764210282094}],"Pan":[0.6897932954008897,0.4093780340332795],"Zoom":5.546093106954236}]}]},
          markerData: [{"﻿String":"ARL13B","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ARL13B&keywords=ARL13B"},{"﻿String":"ASMA","Alias":"A-SMA, a-SMA, alpha-SMA, α-SMA","Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ACTA2&keywords=alpha,smooth,muscle,actin"},{"﻿String":"BANF1","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=BANF1&keywords=BANF1"},{"﻿String":"CD11B","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGAM&keywords=CD11B"},{"﻿String":"CD14","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD14&keywords=CD14"},{"﻿String":"CD163","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD163&keywords=CD163"},{"﻿String":"CD19","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD19&keywords=CD19"},{"﻿String":"CD20","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=MS4A1&keywords=CD20"},{"﻿String":"CD21","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CR2&keywords=CD21"},{"﻿String":"CD3D","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD3D&keywords=CD3D"},{"﻿String":"CD4","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD4&keywords=CD4"},{"﻿String":"CD45","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=PTPRC&keywords=CD45"},{"﻿String":"CD45RB","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=PTPRC&keywords=CD45RB"},{"﻿String":"CD68","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD68&keywords=CD68"},{"﻿String":"CD8A","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD8A&keywords=CD8A"},{"﻿String":"FOXP3","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=FOXP3&keywords=FOXP3"},{"﻿String":"GFAP","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=GFAP&keywords=GFAP"},{"﻿String":"GTUBULIN","Alias":"gamma-tubulin","Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=TUBG1&keywords=gamma,tubulin"},{"﻿String":"IBA1","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=AIF1&keywords=IBA1"},{"﻿String":"KERATIN","Alias":"pan-cytokeratin, pan-keratin","Link":"https://www.genecards.org/Search/Keyword?queryString=KERATIN"},{"﻿String":"KI67","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=MKI67&keywords=KI67"},{"﻿String":"LAG3","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=LAG3&keywords=LAG3"},{"﻿String":"LAMINAC","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=LMNA&keywords=LAMIN,AC"},{"﻿String":"LAMINB","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=LMNB1&keywords=LAMINB"},{"﻿String":"PD-1","Alias":"PD1","Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=PDCD1&keywords=PD-1"},{"﻿String":"PD-L1","Alias":"PDL1","Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD274&keywords=PD-L1"},{"﻿String":"CD19","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD19&keywords=CD19"},{"﻿String":"CD14","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD14&keywords=CD11c"},{"﻿String":"CD56","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=NCAM1&keywords=CD56"},{"﻿String":"CD34","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD34&keywords=CD34"},{"﻿String":"CD44","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD44&keywords=CD34"},{"﻿String":"CD14","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD14&keywords=CD14"},{"﻿String":"CD33","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD33&keywords=CD33"},{"﻿String":"CD41","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGA2B&keywords=CD41"},{"﻿String":"CD61","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGB3&keywords=CD61"},{"﻿String":"CD62","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=SELP&keywords=CD62"},{"﻿String":"CD146","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=MCAM&keywords=CD146"},{"﻿String":"CD1d","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD1D&keywords=CD1d"},{"﻿String":"CD2","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD2&keywords=CD2"},{"﻿String":"CD5","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD5&keywords=cd5"},{"﻿String":"CD7","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD7&keywords=CD7"},{"﻿String":"CD9","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD9&keywords=CD9"},{"﻿String":"CD10","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=MME&keywords=CD10"},{"﻿String":"CD11A","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGAL&keywords=CD11A"},{"﻿String":"CD70","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD70&keywords=CD70"},{"﻿String":"CD74","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD74&keywords=CD74"},{"﻿String":"CD103","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGAE&keywords=CD103"},{"﻿String":"CD133","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=PROM1&keywords=CD133"},{"﻿String":"CD168","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=HMMR&keywords=CD168"}],
          cellTypeData: [{"﻿String":"Natural Killer Cells","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/natural-killer-cells"},{"﻿String":"B Cells","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/cells/b-cells"},{"﻿String":"Basophil","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/basophils"},{"﻿String":"Helper T cell","Alias":"CD4+ T Cell","Link":"https://www.immunology.org/public-information/bitesized-immunology/células/cd4-t-cells"},{"﻿String":"Cytotoxic T Cell","Alias":"CD8+ T Cell","Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/cd8-t-cells"},{"﻿String":"Dendritic Cell","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/cells/dendritic-cells"},{"﻿String":"Eosinophils","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/eosinophils"},{"﻿String":"Macrophage","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/macrophages"},{"﻿String":"Mast Cell","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/mast-cells"},{"﻿String":"Neutrophil","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/cells/neutrophils"},{"﻿String":"Regulatory T Cell","Alias":"Treg","Link":"https://www.immunology.org/public-information/bitesized-immunology/células/regulatory-t-cells-tregs"},{"﻿String":"T follicular helper cell","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/cells/t-follicular-helper-cells"},{"﻿String":"bone marrow","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/%C3%B3rganos-y-tejidos/bone-marrow"},{"﻿String":"lymph node","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/organs-and-tissues/lymph-node"},{"﻿String":"complement system","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/sistemas-y-procesos/complement-system"},{"﻿String":"phagocytosis","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/systems-and-processes/phagocytosis"}],
          homeUrl: "/data/lin-wang-coy-2021",
          speech_bucket: "www.cycif.org"
        });
        document.getElementsByClassName("minerva-legend")[0].style.top = "4em";
        document.getElementsByClassName("minerva-sidebar-menu")[0].style.top = "4em";
    </script>
</body>

</html>
