<!DOCTYPE html>
<html lang="en-US" class="h-100">

<head>
  <meta charset='utf-8'>
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="/assets/css/style.css?v="> <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Graphical Abstract | CyCIF</title>
<meta name="generator" content="Jekyll v3.9.3" />
<meta property="og:title" content="Graphical Abstract" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="A Simple Approach to Highly Multiplexed Cell and Tissue Imaging" />
<meta property="og:description" content="A Simple Approach to Highly Multiplexed Cell and Tissue Imaging" />
<meta property="og:site_name" content="CyCIF" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Graphical Abstract" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","description":"A Simple Approach to Highly Multiplexed Cell and Tissue Imaging","headline":"Graphical Abstract","url":"/data/lin-wang-coy-2021/viz.html"}</script>
<!-- End Jekyll SEO tag -->


</head>

<body class="d-flex flex-column h-100">
    <!-- style="position:relative" -->

  
  

  
  

  
  
  

  
  

    <header class="m-0">
        <nav class="main-nav navbar navbar-expand-lg navbar-dark fixed-top bg-dark mb-md-4">
  <div class="container">
    <a class="navbar-brand" href="/">
      <strong>CyCIF</strong>
    </a>
    <button class="navbar-toggler p-2" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault"
      aria-expanded="false" aria-label="Toggle navigation">
      <span class="text-white">MENU</span>
    </button>
    <div class="collapse navbar-collapse" id="navbarsExampleDefault">
      <ul class="navbar-nav mr-auto">
        
        <li class="nav-item">
          <a href="/data" class="nav-link  active " style="font-size: 1rem">
            Data
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/methods" class="nav-link " style="font-size: 1rem">
            Methods
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/antibodies" class="nav-link " style="font-size: 1rem">
            Antibodies
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/software" class="nav-link " style="font-size: 1rem">
            Software
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/publications.html" class="nav-link " style="font-size: 1rem">
            Publications
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/people.html" class="nav-link " style="font-size: 1rem">
            People
          </a>
        </li>
        
        <li class="nav-item">
          <a href="/funding" class="nav-link " style="font-size: 1rem">
            Funding
          </a>
        </li>
        
      </ul>
          <div class="flex-row ml-md-auto">
  
            <a href="/featured-paper/du-lin-rashid-2019/figures/osd-LUNG_3" class="btn btn-outline-light font-weight-bold" style="font-size: 1rem; background: url(/assets/img/explore_button.jpg) center; background-size: cover;">
              <span class="text-white">
                  Explore CyCIF
              </span> 
            </a>
          </div> 
    </div>
  </div>
</nav>
<nav class="navbar navbar-expand-md navbar-dark bg-dark">
  <div class="container">
    <a class="navbar-brand" href="#">
      <strong>CyCIF</strong>
    </a>
  </div>
</nav>

    </header>

    <div style="height:100%" id="minerva-browser"></div>

    <script defer src="https://use.fontawesome.com/releases/v5.2.0/js/all.js" integrity="sha384-4oV5EgaV02iISL2ban6c/RmotsABqE4yZxZLcYMAdG7FAPsyHYAPpywE9PJo+Khy" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script> 
    <script src="https://api.html5media.info/1.2.2/html5media.min.js"></script>

    <script src="https://cdn.jsdelivr.net/npm/minerva-browser@3.2.0/build/bundle.js"></script>
    <script>
        window.viewer = MinervaStory.default.build_page({
          embedded: true,
          id: "minerva-browser",
          exhibit: {"Name":"Multiplexed 3D atlas of state transitions and immune interactions in colorectal cancer (Minerva Abstract)\n","Header":"Advanced solid cancers are complex assemblies of cancer, immune and stromal cells that invade adjacent tissue. Here we use multiplexed tissue imaging, spatial statistics, and machine learning to identify cell states underlying morphological features of known prognostic significance in colorectal cancer. We find that the necessary spatial analysis requires extended tumor domains, not tissue microarrays or small view-fields. When this condition is met, the data reveal frequent transitions between histological archetypes (tumor grades and morphologies) correlated with molecular gradients. At tumor invasive margins, where tumor, normal, and immune cells compete, localized features in 2D such as tumor buds and mucin pools are seen in 3D to be large connected structures having continuously varying molecular properties. Immunosuppressive cell-cell interactions also exhibit graded variation in type and frequency. Thus, whereas scRNA-Seq emphasizes discrete changes in tumor state, whole-specimen imaging reveals the presence of multi-scale spatial gradients analogous to those in developing tissues.\n","Footer":"","PixelsPerMicron":1.5385,"Images":[{"Name":"WORKAROUND-decouple-storyname-image-description"},{"Description":"CRC01","Height":27120,"MaxLevel":5,"Name":"i0","Path":"https://s3.amazonaws.com/www.cycif.org/lin-wang-coy-2021/data-viz-97","Width":26139}],"Layout":{"Grid":[["i0"]]},"Groups":[{"Channels":["DNA","Pan-cytokeratin","α-SMA","CD4","CD20","CD31"],"Colors":["0000ff","ffff00","ff0000","00ff00","ffffff","ff00ff"],"Name":"Overview 1","Path":"figure1_8__Hoechst2--14__panKeratin--15__aSMA--17__CD4--21__CD20--35__CD31"},{"Channels":["DNA","CD45","E-cadherin","Desmin","CD8"],"Colors":["0000ff","ffffff","00ff00","ff009c","ffbf00"],"Name":"Overview 2","Path":"structure_8__Hoechst2--18__CD45--29__E-Cadherin--34__Desmin--23__CD8a"},{"Channels":["DNA","CD4","FOXP3","PD-1","PD-L1","Pan-cytokeratin"],"Colors":["909090","9600ff","ffff00","00ff00","00ffff","ff0000"],"Name":"Immune supression","Path":"immune-supression_8__Hoechst2--17__CD4--26__FOXP3--19__PD-1--27__PD-L1--14__panKeratin"},{"Channels":["Pan-cytokeratin","E-cadherin","PCNA"],"Colors":["0000FF","00FF00","ff0000"],"Name":"Tumor/Epi only","Path":"figure2-alt_14__panKeratin--29__E-Cadherin--37__PCNA"},{"Channels":["Hematoxylin","Eosin"],"Colors":["8233CC","FF8080"],"Name":"Adjacent H&E","Path":"Histology_40__HE-r--41__HE-g--42__HE-b"},{"Channels":["DNA","Pan-cytokeratin"],"Colors":[898989,"d70000"],"Name":"DNA + Pan-CK","Path":"DNA-CK_8__Hoechst2--14__panKeratin"},{"Channels":["DNA"],"Colors":["898989"],"Name":"DNA","Path":"DNA_8__Hoechst2"}],"Masks":[{"Name":"3 categories","Path":"MASKS/TNP097_3Cats","Colors":["0000FF","00FF00","FF0000"],"Channels":["Tumor/Epi.","Stroma","Immune"]},{"Name":"21 clusters","Path":"MASKS/TNP097_21types","Colors":["1000CB","1600F5","1629F5","2254F5","377DF7","4BA8F8","60D0FA","75FBFD","7AFBD6","89FCB1","A0FC8E","BCFD70","DCFD5B","FEFF54","F8D548","F3AE3D","EF8632","EC622B","EA4325","EA3423","C2281B"],"Channels":["Tumor/Epi.","Ki-67+ Tumor/Epi","PD-L1+ Tumor/Epi","Endothelial","Muscle/Fibroblast","Other","Macrophage (I)","B cell","DN Lymphocyte","T helper","Macrophage (II)","DP Lymphocyte","Treg","PD1+ T helper","Lymphocyte (III)","Macrophage (III)","PD-L1+ Macrophage","Macrophage (V)","Tc cell","PD1+ Tc cell","PD-L1+ Lymphocyte"]},{"Name":"Cluster size grayscale","Path":"MASKS/TNP097_ClusterSize_mono","Colors":["f2f2f2","a6a6a6",333333],"Channels":["Small cluster","Medium cluster","Large cluster"]},{"Name":"Cluster size","Path":"MASKS/TNP097_ClusterSize","Colors":["ff0017","4e00ff","00c7ff","66ff00","ff8700"],"Channels":["Small cluster","S/M cluster","Medium cluster","M/L cluster","Large cluster"]},{"Name":"Predicted morphology","Path":"MASKS/kNN_rgba","Colors":["ff0000","00ff00","0000ff"],"Channels":["Glandular","Solid","Mucinous"]},{"Name":"Annotated tumor buds","Path":"MASKS/TNP097_annBuds","Colors":["ffffff",333333],"Channels":["Tumor buds","Other cells"]},{"Name":"Pathology annotation","Path":"MASKS/fig1_pathlogy_label","Colors":[],"Channels":[]}],"Rotation":0,"Stories":[{"Description":"","Name":"","Waypoints":[{"ActiveMasks":[],"Arrows":[],"Name":"CyCIF profiling of CRC1 sample","Description":"The 24-plex CyCIF (section 097) images from CRC1. Tumor (`Pan-cytoeratin`), stromal (`α-SMA` & `CD31`) and immune (`CD4`/`CD20`) markers are highlighted here, showing distinct tumor, stromal and adjacent normal colon tissue. Tertiary lymphoid structures (TLSs) also appear throughout the specimen, highlighted by `CD20` (white) and `CD4` (green).\n","Group":"Overview 1","Masks":[],"Overlays":[],"Pan":[0.482,0.484],"Zoom":0.5},{"ActiveMasks":["Pathology annotation"],"Arrows":[{"Point":[0.3049931353611071,0.43615868484715237],"Text":"Stromal/muscular","HideArrow":true},{"Point":[0.6421011204293514,0.3110379198378558],"Text":"Adjacent normal colon","HideArrow":true},{"Point":[0.7849403514849437,0.5335050456806337],"Text":"Tumor","HideArrow":true},{"Point":[0.6217715257383356,0.781247673958678],"Text":"Tumor","HideArrow":true},{"Point":[0.14471186752874748,0.7578149404367969],"Text":"Tumor","HideArrow":true}],"Name":"Histopathological view of CRC1 sample","Description":"The resected tumor and adjacent normal tissue in CRC1 is from the cecum of a 69-year old male; pathology revealed a poorly differentiated stage IIIB adenocarcinoma (pT3N1bM0) (Weiser, 2018) with microsatellite instability (MSI-H) and a BRAFV600E (c.1799T>A) mutation. The tumor had an extended front invading into underlying muscle (muscularis propria) and connective tissue giving rise to a \"budding margin\" (A, red-dotted line) adjacent to normal colon epithelium, a \"mucinous margin\" in the middle of the specimen (B, blue-dotted line), and a deep \"pushing margin\" (C, gray-dotted line) \n","Group":"Adjacent H&E","Masks":["Pathology annotation"],"Overlays":[],"Pan":[0.482,0.484],"Zoom":0.5},{"ActiveMasks":["Pathology annotation","3 categories"],"Arrows":[],"Name":"Single-cell data with spatial context 1 - pathological annotation","Description":"Using the MCMICRO software pipeline (Schapiro et al., 2021), images were segmented, fluorescence intensities quantified on a per-cell basis, and cell types assigned using lineage-specific marker expression; a total of ~5 x 10<sup>7</sup> cells were successfully identified in a tissue volume of 1.2 x 10<sup>11</sup> µm<sup>3</sup>. Cells are labeled above by whether their type belongs to a tumor, stroma, or immune category. Overlaying the image with cell type calling allows for analysis of different spatial features and cell-cell interactions in various tumor regions. \n","Group":"Adjacent H&E","Masks":["Pathology annotation","3 categories"],"Overlays":[{"x":0.6524,"y":0.4802,"width":0.1396,"height":0.1005}],"Pan":[0.482,0.484],"Zoom":0.5},{"ActiveMasks":["3 categories"],"Arrows":[{"Point":[0.6911667467179969,0.5188582711076485],"Text":"Tumor/Epithelia","HideArrow":true},{"Point":[0.7454513510307945,0.5355648284525144],"Text":"Immune","HideArrow":true},{"Point":[0.6856428998102303,0.5506432971405159],"Text":"Stromal","HideArrow":true}],"Name":"Single-cell data with spatial context 2","Description":"Cell type calling recapitulate the multiplex CyCIF signals: the dense immune cells inhabit the tertiary lymphoid structures (red) are well separated from the tumor/epithelial compartments (blue) and the surrounding stromal cells (green, endothelial cells & fibroblasts). Note that some cells are excluded due to segmentation errors for which the segmentation algorithm couldn’t effectively separate single cells.\n","Group":"DNA","Masks":["3 categories"],"Overlays":[{"x":0.6524,"y":0.4802,"width":0.1396,"height":0.1005}],"Pan":[0.7329,0.5303],"Zoom":4.6372},{"ActiveMasks":[],"Arrows":[{"Point":[0.7289638792595,0.5160393768280793]},{"Point":[0.7274416553069968,0.5216228237581101]},{"Point":[0.731762571334893,0.5231570014619463]},{"Point":[0.7327488685468048,0.5199419123665427]},{"Point":[0.7467603816849023,0.5303800194694207],"Text":"Tertiary lymphoid structure","HideArrow":true},{"Point":[0.7165745043873903,0.5202005141190915],"Text":"Tumor","HideArrow":true}],"Name":"Tumor buds 1","Description":"Zooming into one of the TLS near the tumor and normal tissue junction, we notice small clusters of tumor cells, known as “tumor buds”. For diagnostic purposes, tumor buds are defined as clusters of ≤4 tumor cells along the invasive front (Lugli et al., 2017b), and less commonly the non-marginal \"internal\" stroma of the tumor mass (Lugli et al., 2011). Tumor buds are thought to associate with tumor metastasis as well as other aggressive phenotypes, but their relationship with immunosurveillance is not fully understood. \n","Group":"Overview 1","Masks":["Annotated tumor buds"],"Overlays":[],"Pan":[0.728,0.5208],"Zoom":16.6158},{"ActiveMasks":[],"Arrows":[{"Point":[0.7289638792595,0.5160393768280793]},{"Point":[0.7274416553069968,0.5216228237581101]},{"Point":[0.731762571334893,0.5231570014619463]},{"Point":[0.7327488685468048,0.5199419123665427]}],"Name":"Tumor buds 2","Description":"In solid tumors, an EMT-like transition at the invasive front gives rise to small clusters of “dedifferentiated” tumor buds likely involved in metastatic dissemination (Zlobec and Lugli, 2010). Lower `E-cadherin` expression is observed in the tumor buds. They also seem to be evading immunosurveillance, e.g. of the nearby `CD8`<sup>+</sup> effector T cells. \n","Group":"Overview 2","Masks":["Annotated tumor buds"],"Overlays":[],"Pan":[0.728,0.5208],"Zoom":16.6158},{"ActiveMasks":[],"Arrows":[{"Point":[0.7289638792595,0.5160393768280793]},{"Point":[0.7274416553069968,0.5216228237581101]},{"Point":[0.731762571334893,0.5231570014619463]},{"Point":[0.7327488685468048,0.5199419123665427]}],"Name":"Tumor buds 3 - immune supression","Description":"A large abundance of T<sub>reg</sub> (`CD4`<sup>+</sup> and `FOXP3`<sup>+</sup>) are also present in this area, alongside expression of the immune checkpoint proteins `PD-1` and `PD-L1`.\n","Group":"Immune supression","Masks":["Annotated tumor buds"],"Overlays":[],"Pan":[0.742,0.53],"Zoom":13.8465},{"ActiveMasks":["21 clusters"],"Arrows":[{"Point":[0.7289638792595,0.5160393768280793]},{"Point":[0.7274416553069968,0.5216228237581101]},{"Point":[0.731762571334893,0.5231570014619463]},{"Point":[0.7327488685468048,0.5199419123665427]}],"Name":"Tumor buds 4 - immune census with spatial context","Description":"Highly multiplexed CyCIF data allow further dissection of cells into different subtypes, including 3 tumor/epi types, 3 stroma cell types and 15 different immune subpopulations. Thus, we can enumerate all these cell types in different tumor regions, and evaluate compositions and potential cell-cell interactions with spatial context. \n\nFor example, we observe an enrichment of immunosuppression subtypes (e.g. T<sub>reg</sub>, PD-L1 macrophage and `PD-1`<sup>+</sup> cells) in close-proximity of budding tumor. \n\n![immunosuppression](/assets/img/lin-wang-coy-2021/immunosupression.png)\n","Group":"DNA","Masks":["21 clusters","Annotated tumor buds"],"Overlays":[],"Pan":[0.742,0.53],"Zoom":13.8465},{"ActiveMasks":["Annotated tumor buds"],"Arrows":[],"Name":"Tumor buds 5 - summary","Description":"Multiple immunosuppressive interactions have been described in colorectal cancer involving ingress of Tregs and expression of checkpoint proteins on both tumor and immune cells.  The established connection between tumor budding, metastasis and disease recurrence is likely to reflect the presence of tumor cells undergoing EMT (and developing the ability to disseminate) as well as a permissive and immunosuppressive environment.\n","Group":"Immune supression","Masks":["Annotated tumor buds"],"Overlays":[],"Pan":[0.7358,0.5184],"Zoom":5.5646},{"ActiveMasks":["Cluster size"],"Arrows":[],"Name":"Tumor cluster size 1","Description":"Instead of separating tumor cells into the discrete category of “buds”, we also quantified the cluster size of all epithelial cells using Delaunay triangulation to estimate adjacency relations between cells.\n","Group":"Adjacent H&E","Masks":["Cluster size","Cluster size grayscale"],"Overlays":[{"x":0.6801,"y":0.5645,"width":0.1318,"height":0.0692},{"x":0.6122,"y":0.6476,"width":0.0573,"height":0.0373}],"Pan":[0.482,0.484],"Zoom":0.5594},{"ActiveMasks":["Cluster size grayscale"],"Arrows":[{"Point":[0.6956075768325325,0.5697874721854945],"Text":"Large cluster, High PCNA, High E-cadherin","HideArrow":true},{"Point":[0.6277176880705782,0.6525386423824076],"Text":"Small cluster, Low PCNA, Low E-cadherin","HideArrow":true}],"Name":"Tumor cluster size 2 - cluster size is related to its expression profile","Description":"We found that expression varied gradually with cluster size, with the expression profile of buds (e.g. low `PCNA` and low `E-cadherin`) representing the extreme of an expression-cluster size continuum. \n","Group":"Tumor/Epi only","Masks":["Cluster size","Cluster size grayscale"],"Overlays":[{"x":0.6801,"y":0.5645,"width":0.1318,"height":0.0692},{"x":0.6122,"y":0.6476,"width":0.0573,"height":0.0373}],"Pan":[0.6849,0.6249],"Zoom":3.1009},{"ActiveMasks":["Predicted morphology"],"Arrows":[],"Name":"Predicting tumor morphology using expression data 1","Description":"A kNN classifier was trained to predict other archetypal features of tumor morphology (glandular, solid, mucinous), using only the CyCIF marker expression of individual epithelial cells. The classifier outputs the probability of a cell belonging to a region of each morphological archetype, and the relative proportion of each archetype is represented by hue.\n","Group":"DNA","Masks":["Predicted morphology"],"Overlays":[{"x":0.6707,"y":0.6605,"width":0.0951,"height":0.0545}],"Pan":[0.482,0.484],"Zoom":0.5594},{"ActiveMasks":["Predicted morphology"],"Arrows":[],"Name":"Predicting tumor morphology using expression data 2 - validation using adjacent H&E section","Description":"The classifier predicts various transition regions between the three tumor morphologies. For example, here, the classifier expects the left-region to be mucinous (blue), transitioning to glandular (red) in the center, and patches of solid (green) towards the top. The adjacent H&E section supports the classifier predictions: contiguous patches of blue cells are indeed in close proximity to mucin, groups of red cells are often part of gland-like structures, while contiguous patches of green cells form solid sheets. Intermediate hues and mixed-color patches of cells suggest intermediate morphologies along a transition gradient.\n","Group":"Adjacent H&E","Masks":["Predicted morphology"],"Overlays":[{"x":0.6707,"y":0.6605,"width":0.0951,"height":0.0545}],"Pan":[0.7197,0.6858],"Zoom":5.3673},{"ActiveMasks":[],"Arrows":[],"Name":"Summary","Description":"Advanced solid cancers are complex assemblies of cancer, immune and stromal cells that invade adjacent tissue. Here we use multiplexed tissue imaging, spatial statistics, and machine learning to identify cell states underlying morphological features of known prognostic significance in colorectal cancer. We find that the necessary spatial analysis requires extended tumor domains, not tissue microarrays or small view-fields. When this condition is met, the data reveal frequent transitions between histological archetypes (tumor grades and morphologies) correlated with molecular gradients. At tumor invasive margins, where tumor, normal, and immune cells compete, localized features in 2D such as tumor buds and mucin pools are seen in 3D to be large connected structures having continuously varying molecular properties. Immunosuppressive cell-cell interactions also exhibit graded variation in type and frequency. Thus, whereas scRNA-Seq emphasizes discrete changes in tumor state, whole-specimen imaging reveals the presence of multi-scale spatial gradients analogous to those in developing tissues.\n\n##### Acknowledgements\nWe thank Alyce Chen, Raquel Arias-Camison, Zoltan Maliga and Jeremy Muhlich and for help in all stages of this project, and Hung-Yi Wu for image registration used in this Minerva story. This work was supported by NIH grants U54-CA225088 (PKS, SS), U2C- CA233262 (PKS, SS), U54-CA225088 (PKS, SS), U2C- CA233291 (CNH, KSL), R01-DK103831 (CNH, KSL), NIH training grant T32-GM007748 (SC)) and the Ludwig Center at Harvard (PKS, SS). All HTAN consortium members are named at (humantumoratlas.org). We thank Dana-Farber/Harvard Cancer Center for the use of the Specialized Histopathology Core, which provided histopathology services supported by P30-CA06516.\n","Group":"Overview 1","Masks":[],"Overlays":[],"Pan":[0.482,0.484],"Zoom":0.5}]}]},
          markerData: [{"﻿String":"ARL13B","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ARL13B&keywords=ARL13B"},{"﻿String":"ASMA","Alias":"A-SMA, a-SMA, alpha-SMA, α-SMA","Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ACTA2&keywords=alpha,smooth,muscle,actin"},{"﻿String":"BANF1","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=BANF1&keywords=BANF1"},{"﻿String":"CD11B","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGAM&keywords=CD11B"},{"﻿String":"CD14","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD14&keywords=CD14"},{"﻿String":"CD163","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD163&keywords=CD163"},{"﻿String":"CD19","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD19&keywords=CD19"},{"﻿String":"CD20","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=MS4A1&keywords=CD20"},{"﻿String":"CD21","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CR2&keywords=CD21"},{"﻿String":"CD3D","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD3D&keywords=CD3D"},{"﻿String":"CD4","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD4&keywords=CD4"},{"﻿String":"CD45","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=PTPRC&keywords=CD45"},{"﻿String":"CD45RB","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=PTPRC&keywords=CD45RB"},{"﻿String":"CD68","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD68&keywords=CD68"},{"﻿String":"CD8A","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD8A&keywords=CD8A"},{"﻿String":"FOXP3","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=FOXP3&keywords=FOXP3"},{"﻿String":"GFAP","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=GFAP&keywords=GFAP"},{"﻿String":"GTUBULIN","Alias":"gamma-tubulin","Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=TUBG1&keywords=gamma,tubulin"},{"﻿String":"IBA1","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=AIF1&keywords=IBA1"},{"﻿String":"KERATIN","Alias":"pan-cytokeratin, pan-keratin","Link":"https://www.genecards.org/Search/Keyword?queryString=KERATIN"},{"﻿String":"KI67","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=MKI67&keywords=KI67"},{"﻿String":"LAG3","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=LAG3&keywords=LAG3"},{"﻿String":"LAMINAC","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=LMNA&keywords=LAMIN,AC"},{"﻿String":"LAMINB","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=LMNB1&keywords=LAMINB"},{"﻿String":"PD-1","Alias":"PD1","Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=PDCD1&keywords=PD-1"},{"﻿String":"PD-L1","Alias":"PDL1","Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD274&keywords=PD-L1"},{"﻿String":"CD19","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD19&keywords=CD19"},{"﻿String":"CD14","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD14&keywords=CD11c"},{"﻿String":"CD56","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=NCAM1&keywords=CD56"},{"﻿String":"CD34","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD34&keywords=CD34"},{"﻿String":"CD44","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD44&keywords=CD34"},{"﻿String":"CD14","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD14&keywords=CD14"},{"﻿String":"CD33","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD33&keywords=CD33"},{"﻿String":"CD41","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGA2B&keywords=CD41"},{"﻿String":"CD61","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGB3&keywords=CD61"},{"﻿String":"CD62","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=SELP&keywords=CD62"},{"﻿String":"CD146","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=MCAM&keywords=CD146"},{"﻿String":"CD1d","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD1D&keywords=CD1d"},{"﻿String":"CD2","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD2&keywords=CD2"},{"﻿String":"CD5","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD5&keywords=cd5"},{"﻿String":"CD7","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD7&keywords=CD7"},{"﻿String":"CD9","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD9&keywords=CD9"},{"﻿String":"CD10","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=MME&keywords=CD10"},{"﻿String":"CD11A","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGAL&keywords=CD11A"},{"﻿String":"CD70","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD70&keywords=CD70"},{"﻿String":"CD74","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=CD74&keywords=CD74"},{"﻿String":"CD103","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=ITGAE&keywords=CD103"},{"﻿String":"CD133","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=PROM1&keywords=CD133"},{"﻿String":"CD168","Alias":null,"Link":"https://www.genecards.org/cgi-bin/carddisp.pl?gene=HMMR&keywords=CD168"}],
          cellTypeData: [{"﻿String":"Natural Killer Cells","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/natural-killer-cells"},{"﻿String":"B Cells","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/cells/b-cells"},{"﻿String":"Basophil","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/basophils"},{"﻿String":"Helper T cell","Alias":"CD4+ T Cell","Link":"https://www.immunology.org/public-information/bitesized-immunology/células/cd4-t-cells"},{"﻿String":"Cytotoxic T Cell","Alias":"CD8+ T Cell","Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/cd8-t-cells"},{"﻿String":"Dendritic Cell","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/cells/dendritic-cells"},{"﻿String":"Eosinophils","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/eosinophils"},{"﻿String":"Macrophage","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/macrophages"},{"﻿String":"Mast Cell","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/c%C3%A9lulas/mast-cells"},{"﻿String":"Neutrophil","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/cells/neutrophils"},{"﻿String":"Regulatory T Cell","Alias":"Treg","Link":"https://www.immunology.org/public-information/bitesized-immunology/células/regulatory-t-cells-tregs"},{"﻿String":"T follicular helper cell","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/cells/t-follicular-helper-cells"},{"﻿String":"bone marrow","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/%C3%B3rganos-y-tejidos/bone-marrow"},{"﻿String":"lymph node","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/organs-and-tissues/lymph-node"},{"﻿String":"complement system","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/sistemas-y-procesos/complement-system"},{"﻿String":"phagocytosis","Alias":null,"Link":"https://www.immunology.org/public-information/bitesized-immunology/systems-and-processes/phagocytosis"}],
          homeUrl: "/data/lin-wang-coy-2021",
          speech_bucket: "www.cycif.org"
        });
        document.getElementsByClassName("minerva-legend")[0].style.top = "4em";
        document.getElementsByClassName("minerva-sidebar-menu")[0].style.top = "4em";
    </script>
</body>

</html>
